Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/unfettered 0 prologue.txt

			choice
				`	"Of course. What do you need me to do?"`
					goto action
				`	"What do you mean, brothers and sisters from Hai space? Are you not different?"`
					goto brethren
				`	"Why do you need me to do it?"`
			label whyme
			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
			choice
				`	"Of course. What do you need me to do?"`
					goto action
				`	"This seems like a bad idea. Ask someone else."`
					decline
			label brethren

> 			`	"Then you have heard about the stories of the Drak modifying our people..." He sighs and deflates a bit. "I do not believe in such things. What makes a Hai - or any other species - “true” is in his heart. It will take him wherever he wants to go." He pauses, thinking, and when he speaks again it is with deep conviction. "Even if we war with the Elders, it does not mean we hate other Hai, nor do they hate us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the limits of the Elders."`

Reviewer: ```suggestion
			`	"Then you have heard about the stories of the Drak modifying our people..." He sighs and deflates a bit. "I do not believe in such things. What makes a Hai - or any other species - 'true' is in his heart. It will take him wherever he wants to go." He pauses, thinking, and when he speaks again it is with deep conviction. "Even if we war with the Elders, it does not mean we hate other Hai, nor do they hate us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the limits of the Elders."`
```
This uses ASCII characters.

			choice
				`	"Let's go then!"`
					goto leave
				`	"Why can't you just use a jump drive and go there yourself?"`
				`	"What will my cut be?"`
					goto cut
			`	"Jump drives be a fine spot of loot amongst me people. I would not risk one by traveling through hostile space. Ye've had one yerself, so ye should know about the scurvy dogs to the south who give no quarter. The lily-livered scallywags to the north would like to see me walk the plank as well... A jump drive would be of no special help to us on this voyage." He seems disappointed by you having to ask such an obvious thing.`
			choice
				`	"Let's go then!"`
					goto leave
				`	"Sorry, but I won't risk my ship by going anywhere close to these pirates."`
					decline
			label cut
			`	"Ye humans always be obsessed with yer doubloons... So many much greater a variety of booty ye could be seeking. Rest assured, it be quite the coffer. I be taking a mere quarter of the total bounty, and ye should well remember how well ye were paid for the ones ye've already delivered to me brethren."`
			label leave

> 			`	"One last warning for ye: Ye should sail the skies with a hyper drive, especially with the dangers of where we be headed. A  jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk the Wanderers decided to fit ye with a hempen halter from yer association with us." You accompany him to your ship, and he seems actually impressed, even more so when seeing the interior. "Your ship indeed be... adequate, for a warrior and this task," he begrudges.`

Reviewer: ```suggestion
			`	"One last warning for ye: Ye should sail the skies with a hyper drive, especially with the dangers of where we be headed. A jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk the Wanderers decided to fit ye with a hempen halter from yer association with us." You accompany him to your ship, and he seems actually impressed, even more so when seeing the interior. "Your ship indeed be... adequate, for a warrior and this task," he begrudges.`
```

				accept
	on visit
		dialog "You do not have the jump drives. You either did not go to <destination> to get the jump drives yet, or you left your escort allocated to carrying the jump drives behind - better launch and wait for it."
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	to complete
		has "Unfettered: Jump Drive Retrieval: offered"
	on complete
		outfit "Jump Drive" -4
		payment 3000000
		log `Helped a young Unfettered Captain to bring jump drives from a pirate world.`
		log "People" "Si-Ra" `A young, impetuous Unfettered captain with a unique, if eccentric, grasp of your language, who seems to grow wiser by the day. He has impressive leadership and combat skills, yet always seeks to refine his tactics. He does seem to be also be a fan of acquiring bigger and bigger ships.`
		conversation

> 			`"Here be yer share: <payment>. Ye have saved me life, Captain <first>, and I shall never forget it. For this, ye have earned the right to know my name: "Si-Ra". It be one I earned and wear with pride. Come see me again soon if ye be up for a challenge. I'll surely be able to find one for ye  even if me own ship not be ready to give ye one. In the meantime, make sure to check the job boards if ye just be looking for more runs like this one."`

Reviewer: ```suggestion
			`"Here be yer share: <payment>. Ye have saved me life, Captain <first>, and I shall never forget it. For this, ye have earned the right to know my name: "Si-Ra". It be one I earned and wear with pride. Come see me again soon if ye be up for a challenge. I'll surely be able to find one for ye even if me own ship not be ready to give ye one. In the meantime, make sure to check the job boards if ye just be looking for more runs like this one."`
```

				goto introduce
			label dosmth
			`	"Of course you can! Even as we speak many Hai are bringing food to our brethren, and so can you. Due to the exceptional circumstances jobs are available to bring more food than ever, for a price corresponding to the seriousness of the situation."`
			label introduce
			`	"Only now I realize I have failed to introduce myself... My name is Vai'chi, and I will probably forever be in your debt. May the leaves keep on flowing your way, and I hope we see each others again. In the meantime, you should know that the Unfettered will probably be busy for some time because of this food crisis."`
			branch over
				not "unfettered: warned the hai governor escort about the mercenary attack"
			`	"My security detachment has told me that you warned them of a great threat. It may be thanks to you that we managed to arrive safely today, and for that I thank you." She then leaves.`
			`	`
			`	Once you get back to your ship you see you received an encrypted message, consisting only of 4 words:`
			`	"Nemo Me Impune Lacessit"`

> 			`	You then notice that you have been fined 1 million credits for 'smuggling of illegal goods' it seems that it is the Unfettered that emited this fine...`

Reviewer: ```suggestion
			`	You then notice that you have been fined 1 million credits for 'smuggling of illegal goods' it seems that it is the Unfettered that emitted this fine...`
```
I warned you

			label questions
			`	"I already have access to the specs of the ship. My questions are mostly related to how well they fare in battle. We've never seriously tried ships carrying fighters before, supposedly because of some ethical implications, but to be honest, it was mostly due to technical difficulties building a ship large enough to contain and launch multiple small ones. I want to know how effective they are and how are they countered." She is clearly very excited to learn more about them.`
			choice
				`	"Well, I've never seen them in combat personally, so you will have to ask someone else. I've only heard stories."`
					goto stories
				`	"That ship was not a proper carrier... wait until you see the some of the Navy's other ships."`
				`	"They're so expensive we don't see too many of them, with the obvious cost of losing fighters all the time... don't bother with them."`
					goto price
			`	"Oh, you mean a ship like the one you scanned, with more fighters and weapons? We already have some scans of that one - exciting scans at that! But do tell me more about it." She seems to be drinking in your words, and is eager for more about those Carriers.`
			choice

> 					`	"Carriers dominate in human space. They have weapons to take down any ship, and with their fighters, they can even take on small fleets."`
> 					`	"Well, I've never seen them in combat personally, so you will have to ask someone else. I've only heard stories."`
> 						goto stories

Reviewer: ```suggestion
				`	"Carriers dominate in human space. They have weapons to take down any ship, and with their fighters, they can even take on small fleets."`
				`	"Well, I've never seen them in combat personally, so you will have to ask someone else. I've only heard stories."`
					goto stories
```

			`	"Arrr, be this the bilge-sucking monkey ye all speak so highly of? The one who fears not to come into our ports, to fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`
			choice
				`	"I fear nothing. Do you?"`
					goto provoke
				`	(Say nothing.)`
				`	(Run away.)`
					decline
			`	"See? The lily-livered monkey dares not even speak! This scallywag be no more than shark bait! Even our 'umble clipper would be sending ye straight to Davy Jones' locker! We be ready to prove it, too, yet we dare not, lest we lose our dancing monkey to the briny deep! But mateys, there be no shame if it dance away, with a dinghy looking like that!" He seems very full of himself. Perhaps someone should teach him a lesson.`
			choice
				`	"You dare insult my <ship>?"`
				`	"I have no need to prove myself to a rodent."`
					goto provoke
				`	(Go. He's not worth it.)`
					decline

> 			`	"Sink me! That flotsam really be your ship? Given the hornswaggle of your reputation, I pegged you for a fancier type. Your prowess is no lie, indeed then, if you land in such a scow, but can ye truly fight in it?" he says, and then he turns to address the crowd once again. "I wonder if it can even launch again. Perhaps we should scuttle it right here."`

Author: @EjoThims any ideas? I'm not the pirate expert ^^

> 			`	"Sink me! That flotsam really be your ship? Given the hornswaggle of your reputation, I pegged you for a fancier type. Your prowess is no lie, indeed then, if you land in such a scow, but can ye truly fight in it?" he says, and then he turns to address the crowd once again. "I wonder if it can even launch again. Perhaps we should scuttle it right here."`

Reviewer 2: This one was a bit deliberately awkward/forced as an attempt to stay in character/theme, but he's blatantly calling the player's reputation a deception without directly calling them a liar themselves. Totally open to appropriate alternatives that work better for that, though.

			choice
				`	"Let me show you my worth in battle. Right now."`
					goto accept
				`	(Crush his rodent face with your fist.)`
				`	(Run away.)`
					decline
			label crush
			`	You rush his way, fist raised, giving his friends in the crowd the exact excuse they were looking for to open fire on you. You wake up later, left alone on the floor. Luckily their weapons only stunned you, but you can forget about doing any further business with the Unfettered.`
				decline
			label accept
			branch "too many ships"
				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 > 12
			`	"Let us prove our worth in battle then, the time for words is over."`
				accept
			label "too many ships"
			`	"Ye cannot run a rig on me, monkey. First ye must park some of those ships! I will not be facing your whole armada. A heavy man-o-war is already over compensating, after all. When ye are ready for a fair duel, see me in orbit."`

> 			`	`

Author: Yeah its to space it out

mission "Unfettered Jump Drive Trading"
	minor
	landing
	invisible
	source "Darkcloak"
	to offer
		has "Unfettered Jump Drive 3: offered"
		not "Wanderers: Jump Drive Source: active"
		random < 15
	on offer
		conversation

> 			`The Unfettered are starting to know you by reputation for bringing jump drives to them. This time, one of them - that looks like an engineer judging by the pad and equipement they have - is waiting for you at your hatch. Maybe they mean to trade something else than money with you. Do you want to bargain another drive with them?`

Author: better?

			choice
				`	(No.)`
					defer
				`	(Yes.)`
			`	You exit your ship and immediately the engineer starts leads you to the outfitter, "We will show you the weapon once you arrive. For now you will only be allowed to access one of them, but if you continue to prove of great assistance to us we may give you access to the others." They lead you trough the usual outfitter but then go to an area you never noticed before, probably because you were not meant to. You have to pass trough a check where they let you keep weapons but verify your identity and history.`
			`	You are eventually taken to an underground zone, passing trough what looks like very basic weapon testing. Unfettered are there, doing small modifications and testing them, but never going really far in the tests. Seeing you look, the engineer explains. "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually shoots before Unfettered captain can get their hands on it." Now that they mention it you notice a few captains looking at the tests, and one even requests a weapon to be taken to his ship, apparently satisfied with its performances.`

> 			`	"Welcome to the special section of the outfitter, captain! It is only for our most distinguished guests. Now that you have sold us multiple jump drives, I can tell you that we can offer something way more valuable than money. For now, I can only offer you the mighty tripulse shredder, but eventually we will give you access to all kinds of weapons. 2 tripulse cannons for one jump drive and our sympathy is the deal I propose." They say, presenting the weapons to you. Now that you can take a closer look, you notice these weapons seem fairly similar to the rest of Hai weaponry, but enhanced someway or another. There even are some ion looking weapons, but one is smaller than an ion cannon whilst another one looks like a turreted version of it.`

Author: Just so uk this was in the tripulse shredder PR, but I'll rework it a bit for sure here

> So a few things here don't make a lot of sense.
> 
> 1. Testing in the vicinity of other expensive equipment stores feels like an inherently bad idea.

seems like smth the Unfettered could do, but I'll specify its well shielded

> 2. Underground anything is expensive and labour intensive to build, and also vulnerable to destruction. This feels like a very bad place for weapons testing of any kind when an open field or even a large warehouse would be options.

*nobody* can learn about this, secrecy is the major rule, also there being only one of these seems reasonable to me
the real testing happens in space indeed, they just make sure the weapon wont explode on your face

> 3. Presumably if captains are witnessing tests and taking them on, there must be some kind of system for iterating and classifying changes. This implies registration, and some kind of special trip that would be made. So none of this feels appropriate to what you would see going into a secure part of the outfitter.
>    I would expect instead that a selection of different "test" variants are on display for captains to take and report back on, maybe with videos or information about their distinctions, and there's the "most popular prototype" option with several in stock that might be on offer to the player.

alr, I added that the more they are appreciated, the more it will be developped (and thus produced)

				accept
	on accept
		outfit "Jump Drive" -1
		outfit "Hyperdrive" 1
		"reputation: Hai (Unfettered)" >?= 40
		fail

phrase "Essential Unfettered Mission Fail"
	word
		`You have failed an essential Unfettered mission. You may want to reload your previous save.`

mission "Unfettered: Bring Hai"
	name `Bring True Hai back`
	description `Bring <bunks> Hai from <stopovers> to <origin>.`
	source "Firelode"
	stopover "Snowfeather"
	passengers 10
	blocked "You will need room for <passengers> in order to take the next Unfettered mission."
	to offer
		has "Unfettered Jump Drive 1: offered"
		has "reputation: Hai" >= 0
		or
			not "unfettered: offered bring hai"
			random < 20
	on defer
		set "unfettered: offered bring hai"
	substitutions
		#They're speaking Hai, but so are the Wanderers.
		<firstgroup> "but because it is in Hai you cannot make sense of the words. Even from afar you can see how malnurished the children are."
			not "language: Wanderer"

> 		<firstgroup> `and you connect your translation device to understand what they say. "... and we fear the next season will be even more [rough, hard, taxing]. We need funding to restore the lands; nothing grows on them anymore!"`

Author: Yeah thats why i defined it both ways, I want the unfettered campaign to be doable at any point, hopefully

			has "language: Wanderer"
		<fourthgroup> "He too demands something, but this Hai does not seem to be malnourished at all. Whatever it is he asked, the Chiefs seem bothered by this request."
			not "language: Wanderer"
		<fourthgroup> `"My cousins would like to join me, but they lack the funds to [easily pay, afford] the transport. Would you allow more to join us, and become [True, authentic]?" This seems to bother the Chiefs, and they answer briefly, "We will contact [True, unaltered] ship captains to ask them if they can, but I fear they are indeed [busy, overworked] these days, which is why the prices have gone up."`
			has "language: Wanderer"
		<closing> "He speaks with a conclusive and conciliatory tone to the many Hai still waiting in line for the stage."
			not "language: Wanderer"
		<closing> `"It is late. You would do well to go home and care for [your family, those near you]. Never hesitate to come back to us in the next [cycle, period] should you need help."`
			has "language: Wanderer"
	on offer
		conversation

> 			`Looking outside, you see the sun is setting. You take a moment to admire the breathtaking view of the sun passing trough some clouds of pollution and notice a large, white, square-shaped structure in the distance, perhaps some sort of stadium, which a growing crowd of Unfettered are streaming towards. Exiting your ship, you cannot avoid being caught up in it, almost carried, as they head towards the building. It is grandiose compared to its surroundings, distinguished by its sheer length and pure white appearance. In contrast, most of the surrounding structures appear to be houses or bars, all with more dilapidated appearances.`

Author: I added this: `Ships keep landing and unloading even more Unfettered all over the spaceport, so much so that, ... caught in the crowd`

			choice
				`	(Ignore her.)`
					goto continue
				`	(Go and speak with her.)`
			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`
			choice
				`	"Yes, please."`
					goto translate
				`	"No, but I did not mean to offend you, so I came to excuse myself."`
			`	"Thank you! No offense is taken. I knew humans were not like they often are depicted here..." she trails off without finishing the thought, instead turning back to the proceedings in the stadium as they continue.`
				goto continue
			label translate
			action
				log "Minor People" "Merora" "A young adolescent Hai that came to Unfettered space after losing her parents to pirates. She is handling the loss very well, at least on the surface."

> 			`	"Alright. I am Merora, by the way. I came to Unfettered space after I lost my parents to pirates, just a few weeks ago. I have not yet integrated well, so I know what it feels like to be isolated here." You begin to introduce yourself to her in turn, but barely get your name out before you are both distracted as the proceedings resume. A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as he speaks, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting him as they speak quietly to each other. The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However he cannot manage alone, and the Chief is asking for someone to escort him or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking anything in return. It only needs to be arranged by someone, as the child does not know who to turn to." Progressively, every single Hai stands up and speaks, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`

Author: Its a crowd thing

We're asking if they're willing/able to find accomodations for the child to come
That said I'll change it to most of them

> 			`	"Alright. I am Merora, by the way. I came to Unfettered space after I lost my parents to pirates, just a few weeks ago. I have not yet integrated well, so I know what it feels like to be isolated here." You begin to introduce yourself to her in turn, but barely get your name out before you are both distracted as the proceedings resume. A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as he speaks, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting him as they speak quietly to each other. The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However he cannot manage alone, and the Chief is asking for someone to escort him or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking anything in return. It only needs to be arranged by someone, as the child does not know who to turn to." Progressively, every single Hai stands up and speaks, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`

Author: like, we are social creatures so its like if everyone starts applauding you do as well

> 			`	The sun is almost set, and you can see that the Chiefs are beginning to grow impatient as the next Hai approaches and speaks to them. Merora tells you, "This request bothers the Chiefs quite a bit, as he is not asking for a child to be transported but a full family, and not from here into Hai space but the opposite. There are not as many captains ready to do this. The Chief said that currently no one is available. Later, they will most probably go and personally ask captains if it can be arranged." While Merora was speaking, the Chiefs had spoken amongst themselves briefly, and now one has moved forward to address the crowd. Merora translates again, "It is over. Goodbye, <first>, I hope we meet again."`

Author: Normally yes but
-there are more Hai willing to come (help etc)
-there are less captains willing to bring them

Or at the very least not enough captains

			label next
			`	As he grows closer, you can tell he is mumbling, presumably to himself, but he speaks in your language, "We do not get alien visitors often here. There has to be a reason for your presence..." Once he has closed the rest of the distance, he speaks again, directly to you and with more focus. "Sorry, I was just organizing my thoughts into your language. Well, you are here anyway, so, maybe, you could help us.`
			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help a friend, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he explains, apparently a bit desperate, considering he knows very little, if anything at all, about you.`
			choice
				`	"Of course. What do you need me to do?"`
					goto action
				`	"What do you mean, brothers and sisters from Hai space? Are you not different?"`
					goto brethren
				`	"Why do you need me to do it?"`
			label whyme

> 			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`

Author: Not the Hai government, but Hai captains that need to make a living, usually tje price was low
But I see what you mean this mission does need work conceptually mmmhhh

> 			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`

Author: I'll add a bit to say that its worse than how it usually is - that said, it is intentional that what he says is biased

		random < 20 + 8 * "Unfettered Jump Drive 4: failed"
	on offer
		conversation
			`As you exit your ship to head to the spaceport you see that an Unfettered Hai<hood> - is waiting for you at your hatch.`

> 			`	As you get closer you summize it is an engineer, judging by the writing pad and equipment they have. Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`

Reviewer 3: ```suggestion
			`	As you get closer you surmise it is an engineer, judging by the writing pad and equipment they have. Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`
```

	on offer
		conversation
			`As you exit your ship to head to the spaceport you see that an Unfettered Hai<hood> - is waiting for you at your hatch.`

> 			`	As you get closer you surmize it is an engineer, judging by the writing pad and equipment they have. Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`

Reviewer: ```suggestion
			`	As you get closer you surmise it is an engineer, judging by the writing pad and equipment they have. Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`
```

			action
				log "Minor People" "Desneeparu" `An Unfettered warrior that knows how to handle a blade. Was tested by her before being given access to new Unfettered weapons. Would have been a goner had she not been holding back.`
			`	She does some kind of salute with her fist before leaving. "I must go now. I hope you will find the weapon you need, Captain."`

> 			`	<The engineer> proceeds to take you underground to what looks like a very basic but very well shielded weapon testing range. They explain, "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually fires before Unfettered captains can get their hands on it. The more interest in a prototype, the more we will develop it." You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`

Author: I prefer not to have the Unfettered reliant on technology to hide their things, but instead on people

because given their conditions, that holographic shield could go down
uk what testing just shouldnt happen here

			choice
				`	"Will this get me into any trouble with the Hai?"`
				`	"What limits?"`
					goto limits
				`	"Why do you need me to do it anyway?"`
					goto whyme
			`	"Quite the contrary! They may not believe in our way of doing things, but they would never dare to restrict the movement of their own people. It would be against all they stand for. Even if we behave differently, they know we are still Hai and that we would never hurt Hai civilians." He seems confident.`
			choice
				`	"What is it exactly that you want me to do?"`
					goto action
				`	"I believe they would not have their place here. I will not bring them."`
					decline
			label limits

> 			`	"I mean that they are prevented from reaching their true potential. Once we had many stars and planets to call ours, but since then most of the Hai have decided to stop changing, to stay peaceful with all species, no matter how primitive. They have chosen to decay, to grow soft. The Unfettered are the ones who choose to keep our legacy, our way of life, and then to evolve, to grow."`

Reviewer 3: Technically they've chosen to re-imagine/reinvigorate/reinstate a legacy way of life - one they re-learned recently from the Korath raids. 
They're not a continuation of an older thing, they're a reboot.

Author: well he just presents it as if the Unfettered were always a thing, resisting the bad moves of the Hai government and such before the raids happened

Reviewer 3: It would be nice if the Player had the opportunity to call him out on it and get that response then. We should have an acknowledgement that their philosophy has justifications for explanations that are otherwise blatantly false on the face of it.

Author: True, I'll make it an option if you have done Hai: ask about the Unfettered

			choice
				`	"What is it exactly that you want me to do?"`
			label action

> 			`	"It's pretty simple: you only need to go to <stopovers>, pick up <bunks> Hai and bring them to this planet. They will find their way from there." You somehow doubt that affirmation, seeing how everything here is in a... ruined state, to say the least. You hope that at least the Hai you're getting are aware of this, or you may have to bring them back...`

Reviewer 3: "ruined" is probably not appropriate.
I imagine the Uhai worlds as chaotic, unstructured, inadequately built and planned for their current capacities.
I wouldn't expect that they're ruined, just overstressed and under managed for a decently long time.

Author: yeah I had dilapidated or smth but smbd wanted to change it, chaotic fits well

			`	When landing on an Unfettered planet, even after having brought them a jump drive, there is always some lingering tension. Now you feel the tension much more than usual, probably because of the medical emergency. Unfettered rush around, and various vehicles are frequently coming and going, bringing both supplies and the infected to the hospital. The hospital itself looks different than the rest of the Unfettered structures: it seems more organized, proper. It is probably the best maintained structure here alongside the shipyards - or at least it has that appearance. One part of it looks to have been grafted onto the rest though, and seems to be a recent addition. Not even their critical buildings appear to be totally safe from the Unfettered's propensity for haphazard building extensions.`
			`	Many pilots seem to have taken up the call, but it appears the organizers here are still interested in all the volunteers they can get. Now that you're here, you're not sure you'll be able to leave without at least being accosted for your assistance with this.`
			choice
				`	(Signal them that I'm going to get the supplies.)`

> 					goto agree

Reviewer 3: If you don't have the language to read the message this choice makes no sense at this point.

This mission offering probably needs a whole branch to it just for investigating and finding out what's going on if you don't have the ability to interpret Hai.

Author: done, I added a thing where you hail a nearby unfettered ship and they somewhat translate the gist of it

			`	You manage to leave without anyone stopping you in all the rush, though you're sure someone was trying to get your attention. If they remember you, the Unfettered will likely not be too happy. You probably also shouldn't stay landed here too long in case they decide to follow you.`
				decline
			label agree
			`	You are quickly assigned the collection of supplies from <stopovers> and urged to return them as quickly as possible.`
				accept
	on stopover
		dialog `You're surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and, once they are done, one of them addresses you. "Tell our brothers and sisters to stay strong. Hopefully it is not too late," he says with determination.`
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit
		dialog phrase "generic missing stopover or cargo"
	on complete
		log `Brought medial supplies to Darkcloak to prevent an epidemic.`
		log "People" "Sookootow" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he has significant influence.`
		payment 1500000
		conversation

> 			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. When you have the time, make sure to check the job board for more jobs like this, it would mean a great deal to me."`

Reviewer 3: So what makes you special for him to single you out? Did you bring more supplies than others, or did he just find it admirable that an outsider would take the time/risk to involve themselves in their affairs?

He should express why it is he's bothered to single you out for thanks in some way.

Author: You're human
And one of the only ones bold and good enough to make it here safely, and the only one to bring a JD

> 			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. When you have the time, make sure to check the job board for more jobs like this, it would mean a great deal to me."`

Reviewer 2: ```suggestion
			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me, if your bravery holds fast, so make sure to check the job board for more jobs like this."`
```

			choice
				`	"I only did what little I could do to help save people. Hopefully I will be able to do more next time we meet."`
					decline
				`	"What could have caused this crisis? Does it happen often?"`
				`	"Are all Unfettered soldiers? I see a lot of you armed and seemingly ready for action."`
					goto soldiers
			`	"It depends on who you ask... Some will say the aliens, your people, are the reason, given the Hai have much contact with them, and we with the Hai. Of course, the Hai who sympathise with our cause do so with many others as well, such as helping humans with low funds relocate to their space. As you have probably noticed, we ourselves do not get many visits from outsiders; they obviously do not feel very welcomed when getting shot at for their supplies. That means we are not protected against the diseases they may bring.`

> 			`	"As for the frequency, the precarious conditions here mean this happens too often for our liking. As a result, we have a habit of quickly reacting to any seemingly deadly diseases. That means that sometimes we worry for nothing, but this was not one of those times. We are a strong people, but even a strong Heshi is nothing in shallow waters. It is a matter of life or death that the hospital be in its best possible state, but it is only that way thanks to much work and investments. The many places in which our people live are not quite as sophisticated."`

Reviewer 3: Just flagging this section; from a medical and disease vector standpoint this whole explanation doesn't really fit with the situation or the question.
It's an isolated section so it doesn't have any impact on anything else, but it should be re-written from scratch at some point. (I'll do it if I get time.)

Author: well it is propaganda that humans bring disease to Hai probably

as for the way to react they do it this way because the fact ppl have low level of health in general (food issues and crammed in buildings) means this will happen often

Author: I think its fine because I don't imagine the Unfettered are competent at dealing with this; at least not him, he is no specialist of disease propagations
mb he could mention that, or you could say smth like before, ~~but you're just a textile worker~~

			choice
				`	"What is a 'Heshi'?"`
				`	"Glad to have prevented a disaster then."`
					decline

> 			`	"The Heshi is a great predator which rules the seas on our homeworld. It is swift and nimble, so you do not see it coming, and when you do, it is already too late. They are not very big, however, so they often hunt in groups. You would translate it as 'sea scorpion', I believe."`

Reviewer 3: ```suggestion
			`	"The Heshi is a great predator which rules the seas on our homeworld. It is swift and nimble, so you do not see it coming, and when you do, it is already too late. They are not very big, however, so they often hunt in groups. Your people have described it as looking like a 'sea scorpion', I believe."`
```

				decline
			label soldiers

> 			`	He looks around him, before answering, "Do I look like a soldier to you? Does this medic look like one? No, we Unfettered only dare to think in another way. In truth, quite a few of us would like to be artists! It is a shame we have to fight for our right to live another way than the one the Hai allow, to break out of the prison they have cast next to our systems! Our soldiers indeed suffer the most from that fight."`

Reviewer 3: ```suggestion
			`	He looks around him, before answering, "Do I look like a soldier to you? Does this medic look like one? No, we Unfettered only dare to think in another way. In truth, quite a few of us would like to be artists! It is the principle of the matter which drives us though, and it is a shame we must help fight for our right to live by another philosophy than the Hai one of stagnation, to break out of the prison they have cast for all of us! Our soldiers indeed suffer the most from that fight."`
```

			
			label ship
			`	As soon as your ship arrives, you give the orders to collect and either transfer the jump drives into cargo or install them directly onto your ship if possible, so you may flee more swiftly. When your ship takes off, you notice more activity than on the way in...`
			`	The Hai comes to speak to you on the bridge. "Batten down the hatches, matey! It took us too long to check and load the jump drives, and we seem to have attracted attention. I guess a lookout got away from the vault and contacted some friends..." Multiple alarms start going off, confirming his suspicions - hostile ships are launching and heading straight for you. You launch as soon as you can, rushing through the correct procedures. Meanwhile, the Unfettered captain joins you on the bridge, clearly used to being where the action is, invited or not.`
				launch
	on accept
		"outfit" "Jump Drive" 4
	# a side mission will spawn additional ships if you brought a JD with you before the start of this mission.
	npc evade
		government "Cerberus"
		personality disables plunders vindictive
		system "Almaaz"
		fleet "Large Northern Pirates"
		fleet "Small Northern Pirates" 2

> # SPOILERS: the tsunami was caused by the Alphas as a warning to listen to their demands (and also because the Unfettered allowed you to see them)

Reviewer 3: Um, what?
Causing a tsunami is no trivial matter

Author: make a crevasse, bam
that said they could have just caused any other disaster

Author: honestly since they have access to Korath tech I think just saying its a disaster can work; Koraths destroyed entire planets
(also it could always be natural cause too, I'm just writing what I had in mind when writing this)

				`	"Seems good to me. I'll do it."`
					accept
				`	"What kind of ship are we talking about?"`
			`	She seems anxious to tell you. "It's a human ship, and quite a big one. We do not know its designation in human space, but it carries a few small ships and uses alien technologies. You do not need to be able to destroy it; just scan it and then come back here." She did not seem to want to answer the last question, and maybe only answered this one so you would go away. Now she is waiting for an answer. Maybe you should reassure her in your abilities before she starts doubting you and hiring someone else.`
			choice
				`	"I'm not worried about any one ship. I'll do it."`
					accept
				`	"That seems too dangerous to me. Sorry, but I will not be helping you anymore."`
					decline
	waypoint "Hi Yahr"
	on waypoint
		dialog `As you enter the system, you see the ship and identify it as a Navy Auxiliary. Whatever happens, the Unfettered asked you not to destroy it and only scan it - at least for now.`
	npc save "scan cargo"
		government "Alpha"
		personality entering timid staying target
		system "Hi Yahr"
		ship "Alpha Auxiliary (Bringing Supplies)" "Settler"
		conversation

> 			`You have finished scanning the Auxiliary. There can be no doubt: it is controlled by Alphas. As you prepare yourself to exit the system and bring the logs back, you receive a message from the local Unfettered authorities.`

Reviewer 3: Why can there be no doubt? To my knowledge the player has no way of determining that at a distance at all.

Author: It is painted pink, also you just scanned it using the unfettered scanner that marks life forms, presumably

			choice
				`	(Ignore her.)`
					goto continue
				`	(Go and speak with her.)`
			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`
			choice
				`	"Yes, please."`
					goto translate
				`	"No, but I did not mean to offend you, so I came to excuse myself."`
			`	"Thank you! No offense is taken. I knew humans were not like they often are depicted here..." she trails off without finishing the thought, instead turning back to the proceedings in the stadium as they continue.`
				goto continue
			label translate
			action
				log "Minor People" "Merora" "A young adolescent Hai that came to Unfettered space after losing her parents to pirates. She is handling the loss very well, at least on the surface."

> 			`	"Alright. I am Merora, by the way. I came to Unfettered space after I lost my parents to pirates, just a few weeks ago. I have not yet integrated well, so I know what it feels like to be isolated here." You begin to introduce yourself to her in turn, but barely get your name out before you are both distracted as the proceedings resume. A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as he speaks, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting him as they speak quietly to each other. The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However he cannot manage alone, and the Chief is asking for someone to escort him or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking anything in return. It only needs to be arranged by someone, as the child does not know who to turn to." Progressively, most of the Hai present stand up and speaks, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`

Reviewer 3: ```suggestion
			`	"Alright. I am Merora, by the way. I came to Unfettered space after I lost my parents to pirates, just a few weeks ago. I have not yet integrated well, so I know what it feels like to be isolated here." You begin to introduce yourself to her in turn, but barely get your name out before you are both distracted as the proceedings resume. A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as he speaks, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting him as they speak quietly to each other. The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However he cannot manage alone, and the Chief is asking for someone to escort him or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking anything in return. It only needs to be arranged by someone, as the child does not know who to turn to." Progressively, most of the Hai present stand up, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`
```
Speaking would be chaotic and unnecessary here.

			choice
				`	(Signal them that I'm going to get the supplies.)`
				`	(I don't have time for this.)`
					goto leave
			label agree
			`	You are quickly assigned the collection of supplies from <stopovers> and urged to return them as quickly as possible.`
				accept
	on stopover
		dialog `You're surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and, once they are done, one of them addresses you. "Tell our brothers and sisters to stay strong. Hopefully it is not too late," he says with determination.`
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit
		dialog phrase "generic missing stopover or cargo"
	on complete
		log `Brought medial supplies to Darkcloak to prevent an epidemic.`
		log "People" "Sookootow" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he has significant influence.`
		payment 1500000
		conversation

> 			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me, if your bravery holds fast, so make sure to check the job board for more jobs like this."`

Reviewer 3: ```suggestion
			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me if your bravery holds fast, so make sure to check the job board for more jobs like this."`
```

			choice
				`	"It seems easy enough. I'll do it."`
					goto agree
				`	"How do I scan it?"`
					goto scan
				`	"I don't want to get in trouble with the Hai."`
			`	"That should not be a problem, the Hai will not bothered about you scanning their ship, if they even notice it."`
			label design
			`	"We would like to know how they avoided vulnerabilities in the hull of a ship of such size. Extra space on a ship cannot only be used for cargo... I will tell you more about it later, once you bring us the scans. That is, if you are willing of course."`
			choice
				`	"Sure, will do."`
					goto agree
				`	"How do I scan it?"`
			label scan
			`	"You need to use one of our Value Detectors, the information provided by it should be sufficient. If you don't have one, you can install one on Darkcloak, on the way there. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`
			choice
				`	"Sure, will do."`
					goto agree
				`	"I'm not interested in scanning ships for you."`
					decline
			label agree

> 			`	On the way back to your ship, Mera runs up to you. "One... last... thing, captain!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you one of our scanners. It will be placed in your cargo hold if there is no space on your ship."`

Reviewer 2: ```suggestion
			`	On the way back to your ship, Mera runs up to you. "One... last... thing, captain!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you one of our scanners. It will be placed in your cargo hold if there is no space on your ship.`
```
Also drop this one with the new stuff added.

		<firstgroup> `You use your translator device to make sense of what is said. "... and we fear the next season will be even more [rough, hard, taxing]. We need funding to restore the lands; very little grows on them anymore!"`
			has "language: Hai"
		<secondgroup> "He does not seem pleased, speaking with contained anger. You are not sure if he is demanding something, or simply complaining."
			not "language: Hai"
		<secondgroup> `"Things are going [poorly, worse]. In all my life never has it been more [rough, unsustainable]. Never have there been so many in need! Never so little to harvest, worse [housing, sanitation]... Something needs to be done, and quickly! We, and our way of life, are dying."`
			has "language: Hai"
		<thirdgroup> "seemingly heartbroken by this little one's words. Raising his voice for all to hear, he appears to be asking something to the gathered Hai."
			not "language: Hai"
		<thirdgroup> `"The little one asks if he can go to Hai [space, safety]," the Chief explains, then continues. "Unfortunately, as you all know, we cannot send back a child alone, he needs to be [taken care of, protected]. Who will fulfill this duty?"`
			has "language: Hai"

> 		<fourthgroup> "This one is also requesting something, but they do not seem to be malnourished at all. The Chiefs seem bothered by this request."

Author: multiple chiefs

> 		<fourthgroup> "This one is also requesting something, but they do not seem to be malnourished at all. The Chiefs seem bothered by this request."

Reviewer 4: Never mind, I was thinking of thirdgroup.

			`	The soldier moves away, and a tense silence hangs in the air for a moment. The Chiefs turn from the audience to speak among themselves, while mutterings and debate seem to be spreading throughout the crowd. This subject appears to be something for which strong opinions are held. The volume increases as the Chiefs continue their aside, and you begin to notice some of the nearby Hai turning their attention towards you, some even pointing you out. It may be that you are not welcome here. One of the younger Hai to notice you, probably still an adolescent judging by her colors, approaches you directly. She speaks softly, in your language, but clearly enough to be heard over the back and forth murmuring of the crowd. "Hi," she says with a warm, somewhat embarrassed chuckle, "I have noticed you easily among Hai. Do you want me to help translate what they say?"`
			choice
				`	"Yes, thank you."`
					goto translate
				`	"Thank you, but I already understand what I need to."`
					goto polite
				`	"I have no need for your help."`
			`	She seems a bit offended that you do not want her help.`

> 			label polite

Author: uh wdym?

> 			label polite

Reviewer 4: On line 580, you can say that you have no need for her help. It then goes:
She seems a bit offended that you do not want her help. 
label polite
She does not press the matter further and proceeds to move away.

			`	She does not press the matter further and proceeds to move away. Most of the Hai here are with what seems to be families or groups of friends, but you do not see your would-be helper join anyone else as she returns to her previous spot in the shifting audience.`
			choice
				`	(Ignore her.)`
					goto headoff
				`	(Go and speak with her.)`

> 			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, looking at the ground with what seems to be contained sadness, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`

Author: Because you're not, you're just as lost as she is
Because she needs smbd to speak to, as well

> 			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, looking at the ground with what seems to be contained sadness, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`

Reviewer 5: But just because someone is lost doesn't mean you start telling your life story to them because you're lost too. She has no idea if you're likely to tell other people about this stuff. She doesn't know who you are or what you're like.

> 			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, looking at the ground with what seems to be contained sadness, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`

Author: so maybe put this in a later conversation, you mean?
`you ask her why she seems alone` thats putting words in the player's mouth kinda as well, and this mission is big enough

> 			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, looking at the ground with what seems to be contained sadness, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`

Reviewer 5: Yes, I would have her tell you this when you know each other better.

				goto continue
			label translate
			action
				"unfettered: Merora translate" ++
				log "Minor People" "Merora" "A young adolescent Hai that came to Unfettered space after losing her parents to pirates. She is handling the loss very well, at least on the surface."
			`	"Great! I am Merora, by the way." She looks around, noticing the way some people are staring at you. Surprisingly they seem to be staring at her almost as much as they do at you. She explains, "I came to Unfettered space when I had nowhere left to turn to, just a few weeks ago. I don't feel like I belong here yet, or anywhere for that matter. So trust me, I know what it feels like to be isolated here. What about you?" she asks.`
			`	You begin to introduce yourself to her, but barely get your name out before you are both distracted as the proceedings resume. A child, younger - or at least smaller - than any other you've seen on stage so far, approaches the Chiefs. Also alone, the child seems scared and perhaps even lost as they speak, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting them as they speak quietly to each other.`

> 			`	The Chief rises and turns to those watching to share the child's words, and Merora then translates it to you, "The child wishes to return home, to Hai space. However, he cannot do that alone, and the Chief is asking for someone to escort him, or find him a suitable transport. All Hai captains, even those who do not otherwise sympathize with our cause, would easily accept taking a child back to Hai space, without ever asking for anything in return. It only needs to be arranged by someone, as the child does not know who to turn to."`

Author: Isnt that already taken into account earlier?
And even if you do, I wouldnt leave it on if she translates it, to be polite?

			choice
				`	(Rise up.)`
				`	(Wait to see what happens.)`
			`	Progressively, most of the Hai present stand up, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see their child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`
			`	The sun has almost set, and you can see that the Chiefs are beginning to grow impatient as the next Hai approaches and speaks to them. Merora tells you, "This request bothers the Chiefs quite a bit, as he is not asking for a child to be transported but a full family, and not from here to Hai space but the opposite. A lot more money can be required to bring Hai here, because whilst many captains agree to bring us food for free, they prefer not to bring us more Hai, since for some it would mean condemning them to know hunger as well. As such, there are not as many captains ready to do this. The Chief said that currently no one is available, at least not at a suitable price. Later, they will most probably go and personally ask captains if it can be arranged."`

> 			`	While Merora was speaking, the Chiefs had spoken amongst themselves briefly, and now one has moved forward to address the crowd. Merora translates again, "It is over. Goodbye, <first>, I hope we meet again."`

Reviewer 6: ```suggestion
			`	While Merora was speaking, the Chiefs had spoken amongst themselves briefly, and now one has moved forward to address the crowd. Merora turns to you, "It is over. Goodbye, <first>, I hope we meet again."`
```

			choice
				`	"Could you help me translate again some time?"`
				`	(Let her leave.)`
					goto leave

> 			`	She blushes, "I am not even sure I will stay in Unfettered space, but if we meet again, I will try, although I cannot promise you more. Some among us are not too fond of humans; it may be hard to prove your worth here, and, if I do stay, I will not want to lose any status I have acquired by speaking with you if you have not yet done so."`

Author: Right. Ig she'll just do a big smile then

			branch leave
				not "Unfettered Jump Drive 1: offered"
			choice
				`	(Tell her you already sold them a Jump Drive.)"`

> 				`	(Don't mention it.)`

Reviewer 6: ```suggestion
				`	(No problem, I understand.)`
```

Reviewer 5: This doesn't have the same meaning as the original line.

			label continue
			`	A child, younger, or at least smaller, than any other you've seen on stage so far, approaches the Chiefs. All alone, the child seems scared and perhaps even lost as they speak, so softly even those on stage must be straining to hear. One of the Chiefs breaks ranks to kneel by the child, comforting them as they speak quietly to each other. The Chief rises; <thirdgroup> Progressively, most Hai stand up and speaks, the message from each to volunteer clear. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He designates one of the first to stand, and gently sends the child walking over to them.`
			`	The sun has almost set, and you can see that the Chiefs are beginning to grow impatient as the next Hai approaches and speaks to them. <fourthgroup> The Chiefs speak amongst themselves briefly then one moves forward to address the crowd. <closing> At this, the crowd, including the young girl who offered to translate for you, leaves, but the Chiefs linger until the crowd has mostly dispersed. It seems one has noticed you now that the crowd has thinned; he is watching you pensively as he comes your way.`
			label next

> 			`	As he grows closer, you can tell he is mumbling, presumably to himself, but he speaks in your language, "We do not get alien visitors often here. There has to be a reason for your presence..." Once he has closed the rest of the distance, he speaks again, directly to you and with more focus. "Sorry, I was just organizing my thoughts into your language. Well, what matters is that you are here, so, maybe, you could help us.`

Author: right they consider alphas as humans

			choice
				`	"I will get to it right away, then."`
					accept
				`	"I don't want to bring you Hai defectors. Sorry."`
			`	He seems a bit shocked. "Defectors?? If you think that way then it is best you do not bring them indeed."`
	on stopover
		dialog `The small group of Hai are waiting for you in the spaceport as planned, and they seem happy to get on your ship. Now return to <origin> with them.`
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit
		dialog phrase "generic missing stopover or passengers"
	on complete
		clear "unfettered: offered bring hai"
		log `Helped some Hai citizens to join the Unfettered.`
		payment 100000
		dialog `The Unfettered thank you for helping these "True" Hai find their way home, and give you <payment>. After discussing with them, the Unfettered Chief comes back to address you, "If you want to help us again, the hospitals on Darkcloak are in grave need of resupply!" Checking back on the group, you can see they are doing fine and already finding a place to call home. They may not have much, but they seem not to require anything at all, other than to be here.`

> mission "<uhai> 1: Bring Medicine Notification"

Author: in the on complete of the last one

> mission "<uhai> 1: Bring Medicine Notification"

Reviewer 4: Resolved.

> 		dialog `The Unfettered thank you for helping these "True" Hai find their way home, and give you <payment>. After discussing with them, the Unfettered Chief comes back to address you, "If you want to help us again, the hospitals on Darkcloak are in grave need of resupply!" Checking back on the group, you can see they are doing fine and already finding a place to call home. They may not have much, but they seem not to require anything at all, other than to be here.`

Reviewer 6: ```suggestion
		dialog `The Unfettered thank you for helping these "True" Hai find their way home, and give you <payment>. After greeting them, the Unfettered Chief comes back to address you, "If you want to help us again, the hospitals on Darkcloak are in grave need of resupply!" Checking back on the group, you can see they are doing fine and are already finding a place to call home. They may not have much, but they seem not to require anything at all, other than to be here.`
```

	landing
	name `Need for medical supplies`
	description `An Unfettered Chief told you to come help the hospitals on <destination> that desperately need supplies.`
	source "Firelode"
	destination "Darkcloak"
	to offer
		has "<uhai> 0: Bring Hai: done"

mission "<uhai> 1: Bring Medicine"
	landing
	name `Bring medical supplies to <planet>`
	description `The Unfettered hospitals on <origin> are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
	source "Darkcloak"
	stopover "Stonebreak"
	cargo "medicals" 50
	blocked `You need <tons> of cargo space available to carry the medical supplies.`
	deadline 9
	to offer
		has "<uhai> 0: Bring Hai: done"
	substitutions

> 		<prioritymessage> `Unfortunately, because it's in Hai, you can only guess at the meaning out of it, but it is clearly important. You can see the Hai word for "Unfettered" repeated several times, as well as numbers in the hundreds.`

Reviewer 6: ```suggestion
		<prioritymessage> `Unfortunately, because it's in Hai, you can only guess at the meaning of it, but it is clearly important. You can see the Hai word for "Unfettered" repeated several times, as well as numbers in the hundreds.`
```

			choice
				`	"What is a 'Heshi'?"`
					to display
						not "outfit: Heshi"
					goto explainheshi
				`	"A Heshi, like this one you mean?"`
					to display
						has "outfit: Heshi"
				`	"Glad to have prevented a disaster then."`
					decline
			`	You can see the confusion on his face when you show him a picture of the Heshi you won from fighting the Captain, "So you have proven yourself already, impressive. You show great potential! The next time you fight this Captain you will gain his position as per the rules of the duels!`
			label explainheshi
			`	"The Heshi is a great predator which rules the seas on our homeworld. It is swift and nimble, so you do not see it coming, and when you do, it is already too late. Most are not very big, however, so they often hunt in groups. Your people have described it as looking like a 'sea scorpion', I believe, and rumour has it one Captain possesses one of monstrous size."`
				decline
			label soldiers

> 			`	He looks around him, before answering, "Do I look like a soldier to you? Does this medic look like one? No, we Unfettered only dare to think in another way. In truth, quite a few of us would like to be artists! It is the principle of the matter which drives us though, and it is a shame we must help fight for our right to live by another philosophy than the Hai one of stagnation, to break out of the prison they have cast for all of us! Our soldiers indeed suffer the most from that fight.`

Author: he keeps talking afterwards so its a convention to not have this one " to make it clear its the same speaker

> 			`	He looks around him, before answering, "Do I look like a soldier to you? Does this medic look like one? No, we Unfettered only dare to think in another way. In truth, quite a few of us would like to be artists! It is the principle of the matter which drives us though, and it is a shame we must help fight for our right to live by another philosophy than the Hai one of stagnation, to break out of the prison they have cast for all of us! Our soldiers indeed suffer the most from that fight.`

Reviewer 4: Resolved.

			choice
				`	"Glad to have prevented a disaster then."`
					decline
				`	"So you just tricked me into helping the war effort?"`
				`	"Were these supplies for soldiers?"`
					goto forsoldiers
				`	"The raiders?"`
					goto raiders
			`	He does not answer your question directly. "These supplies were indeed used to treat an outbreak of a disease that started among soldiers, but thus prevented a wider crisis that would have affected all of us. Most of the wounds came from battle, that is true, but the 'soldiers' as you say need to be kept well just as everyone else, or else we do all suffer. The Hai would not dare to kill us, of course, but they could destroy our way of life, our spirits. And we need our soldiers to bring us supplies."`
				goto leave
			label forsoldiers
			`	"I do not view any of the Hai as 'soldiers' or otherwise separate. We are all Hai and all deserve to be saved; we do not designate supplies to help some before others."`
				goto leave
			label raiders

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Author: also you'd need a moon

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Reviewer 4: ```suggestion
			`	"A few hundreds of years ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`
```
Say this way?

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Author: ig but I kinda wanted an alien way of measuring time

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Reviewer 4: A few hundred cycles?

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Author: I thought about that but thats the same thing the wands use iirc and not very original in general either

I'd prefer it to be abstract how much time has passed

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Reviewer 4: And then eclipses works, the moon just has to go a lot slower.

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Author: but there's no moon!

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Reviewer 4: Your choice. I don’t have any other ideas.

> 			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`

Reviewer 4: That’s what I was thinking.

			``
			`	"Now land at once, we must officially conclude this duel, matey, else it will be worth no more than an empty chest!"`
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail If Killed"
	on visit
		dialog "You have not yet defeated the Unfettered Captain, and landing like this means you choose to delay the duel to another time."
	on complete
		conversation
			`You land and the Captain joins you once he is done with repairs. He presents what looks like an aquarium as a reward for the duel.`
			`	As you move closer to inspect it you are almost frightened to death by the huge scorpion looking creature rushing towards you.`
			`	There is little time to think as the huge creature gets closer and closer!`
			choice
				`	(Run far away in case it escapes!)`
					goto run
				`	(Leap back to be at a respectable distance!)`
				`	(Stay there.)`
					goto stay
				`	(Shoot it.)`
					goto shoot
				`	(Try to calm it.)`
					goto calm
			`	Surprisingly, no Unfettered look amused by the apparent cowardliness. "Fear is good, so long as it does not control you," one of them says.`
			label calm
			action
				"unfettered rep: Si-Ra" ++

> 			`	You hold out your hand peacefully to try and appease it as it jumps out. Surprisingly, it stops right in front of you, and observes you, its eyes deep as the void. The Unfettered are all gawking at you, impressed by this feat of courage that is seemingly paying off. Then a nearby warrior points their weapons at it, and the scorpion-like creature moves faster than you thought possible, striking you with its tail. Then, everything gets dark.`

Reviewer 6: ```suggestion
			`	You hold out your hand peacefully to try and appease it as it jumps out. Surprisingly, it stops right in front of you, and observes you, it's eyes deep as the void. The Unfettered are all gawking at you, impressed by this feat of courage that is seemingly paying off. Then a nearby warrior points their weapons at it, and the scorpion-like creature moves faster than you thought possible, striking you with it's tail. Then, everything gets dark.`
```

			label end
			action
				log `Defeated an Unfettered Captain in a duel, and took part in a surprisingly scary Unfettered martial ceremony.`
				event "unfettered: JD raid plan ready" 5 15
				set "unfettered: duel ready"
				# there's no way to get these before the duel
				"unfettered: took Si-Ra' stuff" = "outfit (all): Gold Bars" + "outfit (all): Ionic Blaster Prototype" * 2
			`	As he passes near you he quickly speaks, "It'll be a few weeks before I'm ready for more action."`
				accept
			label stay
			`	You stay there, surely impressing all Hai present! However it also means you get hit by the tail of the monster as it jumps out to hit you, and everything gets dark.`
			label poisoned
			`	You hear voices around you, someone screaming for help, but also your recent opponent speaking calmly. "There is no cure for humans," you hear as you sink deeper and deeper, down to the edge of unconsciousness, "there is nothing to be done. This is a fight they will have to win on their own."`

> 			`	Suddenly the world darkens, and you fall. When you open your eyes again, all is vivid, as if you had never seen before. You see the scorpion-like creature that attacked you, clearer than ever, in all its might and with a deep spark of intelligence in its eyes, before it swiftly moves away.`

Reviewer 6: ```suggestion
			`	Suddenly the world darkens, and you fall. When you open your eyes again, all is vivid, as if you had never seen before. You see the scorpion-like creature that attacked you, clearer than ever, in all it's might and with a deep spark of intelligence in it's eyes, before it swiftly moves away.`
```

Reviewer: https://www.merriam-webster.com/grammar/when-to-use-its-vs-its

				to display
					not "Unfettered Small Medicals Rush: done"
					not "Unfettered Big Medicals Rush: done"
					not "<uhai> 1: Bring Medicine: done"
				die
			`	Until, finally you are brought back by that same beam of light, and it is all you can see.`
			choice
				`	(Wake up.)`
				`	(Give up.)`
					die
			`	You wake up in a hospital bed, the Captain is there and speaks very quietly, almost gently, "That was maybe not smart, lad, but it sure was brave! You have what it takes to be one of us, just make sure you don't get killed next time." He then moves to the door behind you.`
				goto end
			label run
			action
				log "Defeated an Unfettered Captain in a duel, but it seems that was not the whole test. The second part involved a monstrous scorpion from which it felt like a good idea to run at the time. The Unfettered did not see it that way, however."
			`	You start running away and hear the Captain speak to his crew, loud enough for you to hear, "The monkey is running away from danger. It would not be fit to join us indeed. It will not fight for us."`
				decline
			label shoot

> 			`	You get your weapon out and shoot the monster as it attempts to rush out of its aquarium. You are not sure if your shot over the aquarium hit it, because of the erupting water that came out your way as the beast emerged, but at least you are still alive!`

Reviewer 6: ```suggestion
			`	You get your weapon out and shoot the monster as it attempts to rush out of it's aquarium. You are not sure if your shot went over the aquarium or hit it, because of all the erupting water that came out as the beast emerged, but at least you are still alive!`
```

				goto reward

event "unfettered: JD raid plan ready"

event "cerberus revealed"
	government "Cerberus"
		remove "display name"

# SPOILER: stealing JD from a vault guarded by the Alpha's indoctrinated pirates: Cerberus
mission "<uhai> 3: Recover Jump Drives"
	name `Recover the jump drives`
	description `Go to the pirate world of <stopovers> and recover 4 jump drives from a vault.`
	source "Darkcloak"
	stopover "Freedom"
	clearance
	passengers 1
	to offer
		has "event: unfettered: JD raid plan ready"
	on offer
		conversation

> 			`As you pass what looks like a bar in the spaceport, you're greeted by the same Hai captain that challenged you earlier. "So we meet again! Arr, if ye be wanting to scuttle me clipper again, it'll take a lot more salt out of ye! Well... once I have her fixed and back, anyway. But that not be why I be here, today," he says, apparently happy to see you. "I have a job for ye!"`

Reviewer 6: ```suggestion
			`As you pass what looks like a bar in the spaceport, you hear a familiar voice shout out. "There ye be!" It's the same Hai captain that challenged you earlier. "So we meet again! Arr, if ye be wanting to scuttle me clipper again, it'll take a lot more salt out of ye! Well... once I have her fixed and back, anyway. But that not be why I be here, today," he says, apparently happy to see you. "I have a job for ye!"`
```

Author: monkey is a way to insult you indirectly by not recognizing you, so maybe I should make him say that if he doesnt like you

Reviewer 6: It felt like a good segue as shortly after you call him "my favorite squirrel" and then this, `He does not seem to react to the insult. He did call you monkey a few times before, so it's only fair for you to return the courtesy` 
Also, at this point I don't think they are friends yet as they are not on a first name basis.

			choice
				`	"What can I do for my favorite squirrel?"`
					goto insult
				`	"How soon will you have that new ship?"`
				`	"Well, just do it with that bigger ship of yours, then. I'm not interested."`
					decline
			`	"Once this job be done, hopefully! It should pay handsomely - 'buying a brand new ship' kind of handsome." Judging from your last encounter, this will probably be a very dangerous mission, but it may well be worth it.`
			choice
				`	"What kind of job is it?"`
					goto job
			label insult
			`	He does not seem to react to the insult. He did call you monkey a few times before, so it's only fair for you to return the courtesy.`
			label job
			`	"We be going to collect a few jump drives! Normally ye would not be able to get such a job, since ye have not earned much among us still, but I have the connection to get it for ye. Ye see, officially this be me mission, and a very lucrative one, at that! On top of that, should we succeed, next time ye may be entrusted with such missions on your own, but don't ye be expecting near the same booty for yerself."`
			choice
				`	"Okay... That just seems too good to be true. What's the catch?"`

> 			`	"Well... the world be called 'Freedom', and that may sound nice, but it be a dangerous human port, full of privateers and swashbucklers. That is why I need yer ship, for transportation beyond Hai space. Ye will also need to save room in yer cargo hold or on your flagship for four jump drives. Be warned: this be no mission for the lighthearted."`

Author: idk, I was told in another PR that he comments shouldn't be doing that

> 			`	"Well... the world be called 'Freedom', and that may sound nice, but it be a dangerous human port, full of privateers and swashbucklers. That is why I need yer ship, for transportation beyond Hai space. Ye will also need to save room in yer cargo hold or on your flagship for four jump drives. Be warned: this be no mission for the lighthearted."`

Reviewer 4: Resolved.

			choice
				`	"Let's go then!"`
					goto leave
				`	"parrots?"`
					goto aliens
				`	"Sorry, but I won't risk my ship by going anywhere close to these pirates."`
					decline
			label cut
			`	"Ye humans always be obsessed with yer doubloons... So many much greater a variety of booty ye could be seeking. Rest assured, it be quite the coffer. I be taking half of the total bounty, and ye should well remember how well ye were paid for the ones ye've already delivered to me brethren."`
			label leave
			`	"One last warning for ye: Ye should sail the skies with a hyperdrive, especially with the dangers of where we be headed. A jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk whether or not the parrots up north would decide to fit ye with a hempen halter from yer association with us."`
			choice
				`	(Show him to your ship.)`
					goto ship
				`	"parrots?"`
			label aliens

> 			`	"The aliens we be raiding, ye swab! And don't be joining that party until we tell you so!"`

Reviewer 6: ```suggestion
			`	"The aliens we be raiding, ye swab! And don't be joining that party until we tells you so!"`
```

			`	"Hold up right there matey, the sea sooner be purple than I'd see ye can fit four drives in yer holds! Ya better make some space!"`
				to display
					"cargo space" < 80
			`	You show him to his bunk but he refuses to go there, "Where ye be sending me? Ye need me on the helm, you scabby sea bass!" You have no choice but to let him sit on the bridge.`
				accept
	on enter Alnitak
		dialog `Seeing you enter this system, the captain speaks up. "Sails down, matey! I'd recommend to stop here before we head into more open seas!" He says with a pat on your shoulder that almost makes you lose your balance.`
	on visit
		dialog "You do not have the jump drives. You either did not go to <destination> to get the jump drives yet, or you left them in planetary storage, or you left the escort carrying the jump drives behind."
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	to complete
		has "<uhai> 3: Jump Drive Vault: offered"
	on complete
		outfit "Jump Drive" -4
		payment 2000000
		set "midas: tracker relay"
		log `Helped an Unfettered Captain to bring jump drives from a pirate world.`

> 		log "People" "Si-Ra" `A young, impetuous Unfettered captain with a unique, if eccentric, grasp of your language, who seems to grow wiser by the day. He has impressive leadership and combat skills, yet always seeks to refine his tactics. He does seem to be also be a fan of acquiring bigger and bigger ships.`

Author: first last, but we cant use first person in it caus weird conventions

> 		log "People" "Si-Ra" `A young, impetuous Unfettered captain with a unique, if eccentric, grasp of your language, who seems to grow wiser by the day. He has impressive leadership and combat skills, yet always seeks to refine his tactics. He does seem to be also be a fan of acquiring bigger and bigger ships.`

Reviewer 4: It gets kind of confusing. Oh well, this is resolved also.

			choice
				`	"Don't you want one extra gun?"`
					goto wannafight
				`	"What about the locals? They don't seem friendly."`
			`	"Those bilge-sucking scum? They would never dare attack a well-equipped swashbuckler like meself. Furthermore, there shouldn't be any around these parts, so far from those hovels." He steps forward, then glances back at you, unprotected and armed only with a laser that seems like a toy in comparison to his. "But maybe ye should come with me, after all."`
				goto choose
			label wannafight
			`	"If the vault indeed be compromised, 'twas not by landlubbers. Those inside will be more dangerous and well-armed than the lily-livered locals. Still, I do not be having to guarantee yer safety to secure me loot, so ye be free to do as ye wish. The defense mechanisms be offline for the moment, so ye would have to fend for yerself. These locals be so young and frail and malnourished that even ye can surely handle yerself against them." The disdain in the last words is palpable.`
			label choose
			choice

> 				`	(Come in.)`

Reviewer 4: ```suggestion
				`	"I’ll come in."`
```

Reviewer 4: You probably want to say something here, because Si-Ra says something right after this choice before it describes you going in the cave.

> 				`	(Stay outside.)`

Reviewer 4: ```suggestion
				`	"I’ll stay outside."`
```

Author: I was thinking FL was discrete about it and dont say anything

Reviewer 4: You’re right. This one should be parentheses, but above should be quotation because of Si-Ra responding.

Reviewer 4: This can be resolved.

					goto outside
			# He would be fine with you dying so he does not have to share to loot...

> 			`	He sighs. "As ye wish, but I do hope ye can hold yer own on the ground as well as in space. I have disabled the automated defenses, in case they have been hijacked. We just have to go in, make sure they recognize us as friendlies, and then reactivate them." You really wonder where the vault could be, until you see the Hai walk right through part of the mountain in front of you. It must be a hologram, disguising the entry! You hurry up to catch up to him.`

Reviewer 4: ```suggestion
			`	He sighs. "As ye wish, but I do hope ye can hold yer own on the ground as well as in space. I have disabled the automated defenses, in case they have been hijacked. We just have to go in, make sure they recognize us as friendlies, and then reactivate them." You really wonder where the vault could be, until you see the Hai walk right through part of the mountain in front of you. It must be a hologram, disguising the entry! You hurry to catch up to him.`
```
Double up.

	npc evade
		government "Cerberus"
		system "Almaaz"
		personality disables plunders vindictive
		fleet "Large Northern Pirates"

# This variant has no reverse thrusters to make it easier to scan it and to attract more attention,
# especially with the keystones.
ship "Geocoris" "Geocoris (Keystones)"
	crew 56
	outfits
		"Hai Tracker Pod" 2
		"Hai Tracker" 112
		"Chameleon Anti-Missile" 3
		"Hai Cuttlefish Jammer" 4
		"Boulder Reactor"
		"Hai Valley Batteries"
		"Hai Diamond Regenerator"
		"Hai Corundum Regenerator"
		"Hai Williwaw Cooling" 4
		`"Biroo" Atomic Thruster`
		`"Biroo" Atomic Steering`
		Hyperdrive
		"Pulse Rifle" 56
	cargo
		outfits
			"Quantum Keystone" 123

mission "<uhai> 4: Scan Geocoris"
	name `Scan the Geocoris`
	description `Scan the cargo of the Hai Geocoris in Ya Hai with the Value Detector, then return to <destination> with the scanner logs. You may abort this, at a cost.`
	source "Firelode"
	waypoint "Ya Hai"
	to offer
		random < 60
		has "event: unfettered: access to Mera's lab"
	on offer
		conversation

> 			`As you wander through the spaceport, you find a market with a limited diversity of goods, although some of it seems to be completely alien, even by Hai standards. You can even spot human food and technology, although those are the only recent items here. Most of it is very ancient, and you doubt everything is functional...`

Reviewer 4: ```suggestion
			`As you wander through the spaceport, you find a market with a limited diversity of goods, although some of it seems to be completely alien, even by Hai standards. You can even spot human food and technology, although those are the only recent items here. Most of it is very ancient, and you doubt everything is functional.`
```
To avoid having two paragraphs in a row end with periods.

			branch JD
				has "Unfettered Jump Drive 1: offered"
			`	"I know you have helped us when we needed it most. Word quickly got around of a human meddling in our business, but most view it in a positive way, such as myself. I could use some of that interference myself, in fact!" She says with a slight movement of the ears.`
				goto job
			label JD
			`	"Someone who brings us something as precious as a jump drive is not quickly forgotten! Now back to our business, if you don't mind.`
			label job
			`	"As you probably already know, the other Hai do not think like us. They believe in trading with all the galaxy in harmony forever, without seeing how it softens them. As such, they have wasted time designing a ship for the sole purpose of transporting goods in great quantities: the Geocoris. The biggest upside is being more competitive than humans in terms of the biggest cargo ship, but that would not have been an issue had they not opened trade so much in the first place! Or at least that is how we used to view it. It now seems to us that the design is... interesting, so we want you to scan it." She seems frustrated about this, or perhaps she simply always speaks this quickly.`

> 			`	"It is pretty much guaranteed there will be one such ship at all times around their home system, Ya-Hai, the center of their trade. For obvious reasons we cannot scan that ship ourselves, and we prefer that the Hai be ignorant of our interest in that design, anyway. Bring us the full scanner logs and you will be well rewarded."`

Reviewer 4: ```suggestion
			`	"It is pretty much guaranteed there will be one such ship at all times around their home system, Ya-Hai, the center of their trade. For obvious reasons we cannot scan that ship ourselves, and anyway we prefer that the Hai be ignorant of our interest in that design. Bring us the full scanner logs and you will be well rewarded."`
```

					to display
						not "unfettered: hesitant question"
				`	"I don't want to get in trouble with the Hai."`
					to display
						has "unfettered: hesitant question"
					goto trouble
				`	"I'm not interested in scanning ships for you."`
					to display
						"unfettered: hesitant question" >= 2
					goto declining
			action
				set "unfettered: hesitant question"
			`	"That is what we will be forced to do if you refuse, but we would prefer this to be kept a secret. So do you agree to do this for us?"`
				goto choose
			label trouble
			action
				"unfettered: hesitant question" ++
			`	"That should not be a problem, the Hai will not bothered about you scanning their ship, if they even notice it," she says with a quick sniffing.`
				goto choose
			label design
			action
				set "unfettered: design question"
			`	"We would like to know how they avoided vulnerabilities in the hull of a ship of such size, and the way cargo can be stored without making the ship defenseless. Extra space on a ship cannot only be used for cargo... I will tell you more about it later, once you bring us the scans."`
				goto choose
			label scan
			action
				set "unfettered: scan question"

> 			`	"You need to use at least one of our Value Detectors, the information provided by it should be sufficient, as it goes beyond a simple cargo scan. If you don't have one, you can install one here. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`

Reviewer 4: ```suggestion
			`	"You need to use at least one of our Value Detectors. The information provided by it should be sufficient, as it goes beyond a simple cargo scan. If you don't have one, you can install one here. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`
```

			action
				outfit "Value Detector" 2
				outfit "Cargo Scanner" 5
				clear "unfettered: scan question"
				clear "unfettered: design question"
				clear "unfettered: hesitant question"
			`	"I cannot stress enough that you NEED to use at least one Value Detector, as any other scanners will simply either not contain enough information, or we could be unable to make good use of it. Since the Geocoris is quite complex, it may take some time to complete the scan, so feel free to buy more scanners here. I'd recommend going to the lab to check if they are well connected to your ship."`
			`	"Do note, however, that they can be supplemented by Cargo Scanners, which is why I have decided to provide a few of those to you as well. You can go to the outfitter to set it all up correctly."`
				accept
			label declining
			action
				"unfettered rep: Mera" --
			`	She clench her fist, visibly angry for a moment, before calming down. "Alright, but it will take a while for me to get someone else to scan a Geocoris, and we're missing a significant opportunity."`
			choice
				`	"Okay, okay. I changed my mind."`
					goto choose
				`	"Not my problem."`
					decline

> 		log "People" "Mera" `An Unfettered engineer who is interested in ship designs of all kinds. She is easily excited by new technologies and ships, and seems to communicate partly through her bodily language.`

Reviewer 4: How does this get put in the log?

Author: you write it, idk

Reviewer 4: What I mean is, what label leads to this happening?

Author: the body language is smth I tried to add a bit everywhere she talks, and as for the ships she's interested enough in the geocoris design to hire a human outsider

Reviewer 4: Yes. How does this go into the log though? Does it do it when you accept or something like that?

Author: it goes into the log when the mission offers, as you get the convo

Reviewer 4: Ok. Then this is resolved.

			action
				fail "<uhai> 4: Scan Geocoris"
	npc
		government "Hai"
		to spawn
			has "unfettered: entered Ya Hai"
		personality launching coward
		system "Ya Hai"
		ship "Grasshopper (Ion)" "Beholder"
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit
		dialog `You have not scanned the Geocoris yet. Go to Ya Hai to scan it, and then come back here.`
	on complete
		"unfettered: Solifuge deployment" ++
		clear "unfettered: entered Ya Hai"
		log `Scanned a Geocoris for the Unfettered. They seemed interested in the design - probably with a ship of their own in mind.`
		payment 150000
		dialog `You send the scans to the Unfettered government and receive a payment of <payment>. They tell you Mera will need some time to analyze the scans and you may return in a few weeks for further work.`
		event "unfettered: analyzed the scans" 10 20

mission "<uhai> 4: Geocoris Arrival"
	landing
	invisible
	source
		system "Ya Hai"
	to offer
		has "<uhai> 4: Scan Geocoris: active"
	on offer
		conversation
			`Shortly after you land, you see the Geocoris signature on your radar. You take off immediately.`

> 				flee

Reviewer 4: Flee makes all ships in the system hostile to you, right? Shouldn’t this be `launch` or `depart`?
```suggestion
				launch
```

Author: Flee launches you and declines the mission, that's all (and there's no point in activating this one mission)

Reviewer 4: Okay.

mission "<uhai> 0: Bring Hai"
	name `Bring True Hai`
	description `Bring <bunks> Hai from <stopovers> to <origin>.`
	source "Firelode"
	stopover "Snowfeather"
	passengers 10
	blocked "You will need room for <bunks> <passengers> in order to take the next Unfettered mission."
	to offer
		"reputation: Hai (Unfettered)" >= 0
		# guaranteed to offer the first time and then 20% chance
		or
			not "unfettered: offered bring hai"
			random < 20
	on defer
		set "unfettered: offered bring hai"
	substitutions
		# Translate it if you have a translator, otherwise someone will offer to do it for you.

> 		<firstgroup> "but because it is in Hai you cannot make sense of the words. However even from afar you can see how malnurished the children are."

Reviewer: ```suggestion
		<firstgroup> "but because it is in Hai you cannot make sense of the words. However even from afar you can see how malnourished the children are."
```

				`	"What limits?"`
					goto limits
				`	"Why do you need me to do it anyway?"`
					goto whyme
			`	"Quite the contrary! They may not believe in our way of doing things, but they would never dare to restrict the movement of their own people. It would be against all they stand for. Even if we behave differently, they know we are still Hai and that we would never hurt Hai civilians." He seems confident when telling you this.`
			choice
				`	"What is it exactly that you want me to do?"`
					goto action
				`	"I believe they would not have their place here. I will not bring them."`
					decline
			label limits
			`	"I mean that they are prevented from reaching their true potential. Once we had many stars and planets to call ours, but since then most of the Hai have decided to stop changing, and to stay peaceful with all species. They have chosen to decay, to grow soft. We Unfettered decided to keep our legacy, our way of life, and to evolve, to grow."`
			choice
				`	"What is it exactly that you want me to do?"`
					goto action
				`	"But I thought the Unfettered only split from the Hai a few hundred years ago?"`
					to display

> 						has "Ask the Hai about the Unfettered: offerred"

Reviewer: ```suggestion
						has "Ask the Hai about the Unfettered: offered"
```
Pretty sure this was just broken

			action
				set "unfettered: heshi trip"
				"unfettered rep: Si-Ra" ++
			`	Except you never wake up. The final words you hear before all is truly dark speak of an ongoing medical crisis taking priority.`
				to display
					not "Unfettered Small Medicals Rush: done"
					not "Unfettered Big Medicals Rush: done"
					not "<uhai> 1: Bring Medicine: done"
				die
			`	Until, finally you are brought back by that same beam of light, and it is all you can see.`
			choice
				`	(Wake up.)`
				`	(Give up.)`
					die
			`	You wake up in a hospital bed, the Captain is there and speaks very quietly, almost gently, "That was maybe not smart, lad, but it sure was brave! You have what it takes to be one of us, just make sure you don't get killed next time." He then moves to the door behind you.`
				goto end
			label run
			action

> 				log "Defeated an Unfettered Captain in a duel, but it seems that was not the whole test. The second part involved a monstruous scorpion from which it felt like a good idea to run at the time. The Unfettered did not see it that way, however."

Reviewer: ```suggestion
				log "Defeated an Unfettered Captain in a duel, but it seems that was not the whole test. The second part involved a monstrous scorpion from which it felt like a good idea to run at the time. The Unfettered did not see it that way, however."
```

			`	Most Unfettered here are just busy buying what little food there is, that is if they can even afford it. You see two of them disputing, probably over the cost of some item, but decide not to intervene given how they generally respond to your mere presence...`
			`	An Unfettered approaches you. At first, you wonder why the Hai happens to be coming towards you in particular, but then you realize that you naturally stand out in the crowd. There are not many non-Hai on the Unfettered worlds.`
			`	"Captain <last>! I am Mera, an engineer. We have an easy job for you, if you are interested," she says. "It would pay well," she adds.`
			choice
				`	"Sure, what can I do for you?"`
				`	"How do you know me?"`
					goto know
				`	"Sorry, but I'm quite busy at the moment."`
					defer
			action
				"unfettered rep: Mera" ++
			`	"Straight to the point! I am starting to like you."`
				goto job
			label know
			branch JD
				has "Unfettered Jump Drive 1: offered"

> 			`	"I know you have helped us when we needed it most. Word quickly got around of a human medling in our business, but most view it in a positive way, such as myself. I could use some of that interference myself, in fact!" She says with a slight movement of the ears.`

Reviewer: ```suggestion
			`	"I know you have helped us when we needed it most. Word quickly got around of a human meddling in our business, but most view it in a positive way, such as myself. I could use some of that interference myself, in fact!" She says with a slight movement of the ears.`
```

				goto leave
			label forsoldiers
			`	"I do not view any of the Hai as 'soldiers' or otherwise separate. We are all Hai and all deserve to be saved; we do not designate supplies to help some before others."`
				goto leave
			label raiders
			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`
			label leave
			`	Someone then calls upon Sookootow in hai and he is forced to promptly leave.`

event "unfettered: access to Mera's lab"
	planet "Firelode"
		add outfitter "Mera's Lab"

outfit "Heshi"
	category "Unique"
	cost 10000
	thumbnail "outfit/unknown"
	"mass" 1
	"capture attack" 4
	"capture defense" 8
	"maintenance costs" 100
	description "This is an aquarium containing what the Hai call a 'Heshi' - a huge scorpion looking creature. It must be fed and cared for but can be very useful in damaging the moral of enemies attacking the ship."

> 	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware they could become its next meal if they do not cooperate... Furthermore, it is very useful in the event of an interrogation."

Author: idk it's more of a campaign one off thing

> 	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware they could become its next meal if they do not cooperate... Furthermore, it is very useful in the event of an interrogation."

Reviewer 7: Oh... is this item supposed to be unique? I got two of them in my playthrough. I plundered a Heshi from the captain's ship, and I got another one during the follow-up conversation. Maybe the one on his ship should be removed? It's defined in `midas.txt`

> 	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware they could become its next meal if they do not cooperate... Furthermore, it is very useful in the event of an interrogation."

Author: silly me forgot the unplunderable

# SPOILER: he tests your military aptitude before asking you to take him to pirate space.
# gameplay wise if you win this mission you should be able to do the following ones, it tests if your strength is high enough.
# Since it is the first one you can do it many times until you succeed.
mission "<uhai> 2: Prove Your Strength"
	autosave
	repeat
	deadline 2
	name `Defeat the captain`
	description `A young Unfettered captain has provoked you into a duel. Disable his ship and then land back on <destination>. You only have one day, but this duel can happen multiple times.`
	source "Darkcloak"
	to offer
		or
			has "<uhai> 2: Prove Your Strength: deferred"
			has "Wanderers: Defend Vara Ke'sok: aborted"
			random < "Unfettered Jump Drive 1: offered" * 20 + "Unfettered Jump Drive 2: offered" * 25 + "Unfettered Jump Drive 3: offered" * 30 + "Unfettered Jump Drive 4: failed" * 25
		# 3 is what a default built Sea Scorpion has
		"armament deterrence" >= 3
		not "<uhai> 2: Prove Your Strength: done"
	on offer
		conversation
			branch waiting
				or
					has "<uhai> 2: Prove Your Strength: deferred"
					has "<uhai> 2: Prove Your Strength: offered"

> 			`You start wandering in the spaceport, and on the way you notice just how much everything around you here is simple, efficient, and ordered, unlike the world of Firelode. The Unfettered present are proudly showing their weapons, more so than on other planets, and some of them are even armored. They seem ready for action, at any time. Thankfully they no longer consider you an enemy, at least for the moment.`

Reviewer 7: This line mentions Firelode, but I encountered this mission on my playthrough without ever having visited Firelode. This was pretty confusing.
<img width="192" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/653f26c4-b928-4d81-958c-3e43ecf27419">

			`	Eventually your attention is drawn to a gathering, where someone is speaking loudly to a small crowd of people. They all seem very familiar with each other, and as you watch, they have noticed you too. The one who was addressing the others saunters through the group and over to you. This could mean trouble.`

> 			`	"Arrr, be this the bilge-sucking <, unfettered title> ye all speak so highly of? The one who fears not to come into our ports, and fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`

Reviewer 7: ```suggestion
			`	"Arrr, be this the bilge-sucking<, unfettered title> ye all speak so highly of? The one who fears not to come into our ports, and fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`
```

> 			`	"Arrr, be this the bilge-sucking <, unfettered title> ye all speak so highly of? The one who fears not to come into our ports, and fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`

Reviewer 2: ```suggestion
			`	"Arrr, be this the bilge-sucking, <unfettered title> ye all speak so highly of? The one who fears not to come into our ports, and fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`
```

This seems more right to me?

Author: <, uhai title> is a substitution I made bc I always need it with the ,

Reviewer 7: Yeah, the name of the substitution is literally `<, uhai title>`, so that wouldn't work unless you changed the name of the substitutions as well.

			choice
				`	"I fear nothing. Do you?"`
					goto provoke
				`	(Say nothing.)`
				`	(Run away.)`
					goto run

> 			`	"See? The lily-livered monkey dares not even speak! This scallywag be no more than shark bait! Even our 'umble clipper would be sending ye straight to Davy Jones' locker! We be ready to prove it, too, yet we dare not, lest we lose our dancing monkey to the briny deep! But mateys, there be no shame if it dance away, with a dinghy looking like that!" He seems very full of himself. Perhaps someone should teach him a lesson.`

Reviewer 2: He's very deliberately talking in human references instead of Hai.

The whole pirate speak thing is a deliberately worn affectation and that wouldn't be nearly as clear if it were adapted to Hai cultural references, especially before he explains that himself.

> 			`	"See? The lily-livered monkey dares not even speak! This scallywag be no more than shark bait! Even our 'umble clipper would be sending ye straight to Davy Jones' locker! We be ready to prove it, too, yet we dare not, lest we lose our dancing monkey to the briny deep! But mateys, there be no shame if it dance away, with a dinghy looking like that!" He seems very full of himself. Perhaps someone should teach him a lesson.`

Reviewer 7: Ah, I haven't gotten to that part yet.

			choice
				`	"You dare insult the <ship>?"`
				`	"I have no need to prove myself to a rodent."`
					goto provoke
				`	(Go. He's not worth it.)`
					goto run
			`	"Sink me! That flotsam really be your ship? Given the hornswoggle of your reputation, I pegged you for a fancier type. Your prowess is no lie, indeed then, if you land in such a scow, but can ye truly fight in it?" he says, and then he turns to address the crowd once again. "I wonder if it can even launch again. Perhaps we should scuttle it right here."`
				goto lesson
			label provoke
			`	"So the space-monkey can speak? Let us see if it can fight as well," he says with a provocative tone.`
			label lesson

> 			`	Time for him to learn a lesson. The thought that you may be in over your head briefly crosses your mind, but you brush it aside.`

Reviewer 2: Fair point.

> 			`	Time for him to learn a lesson. The thought that you may be in over your head briefly crosses your mind, but you brush it aside.`

Reviewer 2: ```suggestion
			`	You may be in over your head here, but it seems time for one of you to learn a lesson.`
```
x

> 			`	Time for him to learn a lesson. The thought that you may be in over your head briefly crosses your mind, but you brush it aside.`

Reviewer 2: ```suggestion
			``
```
Alternatively, we can just clip this and it all flows just as well as it does now.

Author: is "it might be time for him to learn a lesson" fine?

Reviewer 2: Yea, that would work, too, I think.

			`	After a cheering from his crowd, the arrogant Hai then speaks to you directly. "Ye and me then, a battle of flagships. Face me far above the tallest trees, whenever ye find the courage. But rest assured, monkey, I have the skill to humiliate ye without feeding ye to the fishes!"`
			choice
				`	"Let me show you my worth in battle. Right now."`
					goto accept
				`	"Can you tell me more about the rules of this fight?"`
					goto rules
				`	(Crush his rodent face with your fist.)`
					goto crush
				`	(Run away.)`
			label run
			`	As you move away you hear laughter, but nobody tries to stop you. You have just shown to the Unfettered how much of a coward you are, and will probably never be offered any missions from them anymore.`
				decline
			label rules

> 			`	"I guess it only be fair that ye know... Ye're supposed to bring only yer flagship, but if ye admit it be too weak, then more ships can be brought, so long as it still be a fair battle. Do not expect anyone to be fighting ye for honor if ye bring an armada. The challenge lasts until one of our ships is scuttled or one of us agrees to walk the plank. Ye can always stall the call to battle, if ye were to offer a symbolic bribe of 1000 credits, offered in space before the battle begins, to demand it be fought another day. After ye did that, ye must wait a full day, or 'till be considered an offense, and all nearby ships will fire at you! The challenge must eventually be answered though, no matter how long or often ye want to delay it. Finally, it is right of the victor to plunder the ashamed, but absolutely not that ye try to capture the ship. So, be ye ready to face yer doom?"`

Reviewer 2: ```suggestion
			`	"I guess it only be fair that ye know... Ye're supposed to bring only yer flagship, but if ye admit it be too weak, then more ships can be brought, so long as it still be a fair battle. Do not expect anyone to be fighting ye for honor if ye bring an armada. The challenge lasts until one of our ships is scuttled or one of us agrees to walk the plank. Ye can always stall the call to battle, if ye were to offer a symbolic bribe of 1000 credits, offered in space before the battle begins, to demand it be fought another day. After ye did that, ye must wait a full day, or 'twill be considered an offense, and all nearby ships will fire at you! The challenge must eventually be answered though, no matter how long or often ye want to delay it. Finally, it is right of the victor to plunder the ashamed, but absolutely not that ye try to capture the ship. So, be ye ready to face yer doom?"`
```

> 			`	"I guess it only be fair that ye know... Ye're supposed to bring only yer flagship, but if ye admit it be too weak, then more ships can be brought, so long as it still be a fair battle. Do not expect anyone to be fighting ye for honor if ye bring an armada. The challenge lasts until one of our ships is scuttled or one of us agrees to walk the plank. Ye can always stall the call to battle, if ye were to offer a symbolic bribe of 1000 credits, offered in space before the battle begins, to demand it be fought another day. After ye did that, ye must wait a full day, or 'till be considered an offense, and all nearby ships will fire at you! The challenge must eventually be answered though, no matter how long or often ye want to delay it. Finally, it is right of the victor to plunder the ashamed, but absolutely not that ye try to capture the ship. So, be ye ready to face yer doom?"`

Reviewer 2: The added be is redundant without removing the current, and moving it changes the meaning subtly.

> 			`	"I guess it only be fair that ye know... Ye're supposed to bring only yer flagship, but if ye admit it be too weak, then more ships can be brought, so long as it still be a fair battle. Do not expect anyone to be fighting ye for honor if ye bring an armada. The challenge lasts until one of our ships is scuttled or one of us agrees to walk the plank. Ye can always stall the call to battle, if ye were to offer a symbolic bribe of 1000 credits, offered in space before the battle begins, to demand it be fought another day. After ye did that, ye must wait a full day, or 'till be considered an offense, and all nearby ships will fire at you! The challenge must eventually be answered though, no matter how long or often ye want to delay it. Finally, it is right of the victor to plunder the ashamed, but absolutely not that ye try to capture the ship. So, be ye ready to face yer doom?"`

Reviewer 7: Meant to remove the previous `be`, but I'll edit to remove that change and keep the other fixes.

			choice
				`	"Let me show you my worth in battle. Right now."`
					goto accept
				`	(Crush his rodent face with your fist.)`
				`	(Run away.)`
					goto run
			label crush
			`	You rush his way, fist raised, giving his friends in the crowd the exact excuse they were looking for to open fire on you. You wake up later, left alone on the floor. Luckily their weapons only stunned you, but you can forget about doing any further business with the Unfettered.`
				decline
			label waiting
			`Do you want to resume the duel with the Captain now?`
			choice
				`	(Yes.)`
				`	(Later.)`
					defer
			`	You look for the ship of the Captain using its signature, and once you find it proceed to ask his crew to contact him, making your way back to your own ship. He comes surprisingly quickly, and his ship is already in the air above you by the time you can reach yours. He hails you directly.`
			label accept
			branch "too many ships"

> 				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 12

Author: yeah you're not gonna be allowed to duel a sea scorpion with the sword
later its allowed when ur fighting a beetle

> 				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 12

Reviewer 7: If that's true, then you need to change the below dialogue, which tells the player that they have to "park some of those ships". If the player only has the Emerald Sword and no other ships, they will think this is a bug.

			action
				"<uhai> 2: Prove Your Strength: deferred" ++
			`	Land again when you meet those specifications.`
				defer
	on enter
		clear "<uhai> 2: Prove Your Strength: deferred"
		conversation
			branch "cheating bugger"
				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 12
			`The captain launches right after you, and hails you as soon as you enter orbit, probably anxious to get the fight started.`
			`	"I saw fit to bring a bit of extra crew to make it a fair fight, as it is no sitting duck that you brought! Given the fragile nature of their ships, we'll understand if they were to go down in battle. Just do not capture their ship.`
				to display
					"ships: Medium Warship" + "ships: Heavy Warship" * 2 > 1
			`	"May crossing swords sharpen our blades. Let it begin!"`
				accept
			label "cheating bugger"

> 			`The captain hails you, "That is too big of a fleet, ye monkey! This challenge sinked, come back to me again when ye 'r ready to do it by the code! It'll cost ye <fine> for wasting my time."`

Reviewer 2: "Be sunk" instead.

> 			`The captain hails you, "That is too big of a fleet, ye monkey! This challenge sinked, come back to me again when ye 'r ready to do it by the code! It'll cost ye <fine> for wasting my time."`

Reviewer 2: ```suggestion
			`The captain hails you, "That is too big of a fleet, ye monkey! This challenge be sunk! Come back to me again when ye 'r ready to do it by the code! It'll cost ye <fine> for wasting my time."`
```

		personality disables staying heroic target launching unconstrained
		# vulnerable to ion
		ship "Si-Ra' Scorpion" "Intrepid"
		on kill
			conversation "unfettered outcast"
		conversation
			`Now that you have disabled the <npc>, its captain hails you.`
			branch "wanderer first"
				has "Wanderers: Defend Vara Ke'sok: aborted"
			`	"It seems ye fight admirably after all. Ye've earned my respect today," he says with a voice so calm you are surprised it came from the same captain who was recently provoking you for no apparent reason.`
				goto fire
			label "wanderer first"
			`	"Well<, unfettered title>, it seems you be worthy of joining our crew! I was the one who contacted you on behalf of our ship captain, in the alien space up north! Ye've earned my respect today," he says with a voice so calm you are surprised it came from the same captain who was recently provoking you for no apparent reason.`
			`	"However, since I knew you I made sure to allow you to prove yerself pretty fast, but if you want to continue doing business with us, jump drives are always appreciated..."`
				to display
					not "Unfettered Jump Drive 1: offered"
			label fire

> 			`	In the background you hear fires roaring, it seems the damage to his ship was quite serious indeed.`

Reviewer 2: Semicolon would be more appropriate and truer to original imo

> 			`	In the background you hear fires roaring, it seems the damage to his ship was quite serious indeed.`

Reviewer 2: ```suggestion
			`	In the background you hear fires roaring; it seems the damage to his ship was quite serious indeed.`
```

				goto end
			label weapons
			`	"Aye, matey! The flashier yer weapons be when ye run a shot across their bow, the more likely these landlubbers be to strike colors and ditch their loot! No need to make these lily livered into shark food, just shiver their timbers a bit. Savvy? And not much else sets yer knees a knockin' quite like sending watching yer weapons go squiffy while yer energy levels drop straight ta Davy Jones' Locker! Always a good clap o' thunder for 'em!"`
			label end
			action
				payment 250000
			`	"Hopefully we see each other again, sometime." He and his crew seem quite busy with the repairs. "I transferred yer booty of <payment> to pay to repair any damage I may have done to yer ship. I be landing on the surface soon, to drown the sorrow of this humiliating defeat I have just suffered. I do not need nor want yer assistance in repairing, for that is a task best suited for the vanquished. I suppose ye may board me old ship to claim yer right o' salvage on me but ye wouldn't do that to a friend, would ye?"`

> 			`	"If ye be feeling up to the challenge, there be other captains that would be willing to duel ye, if ye keep fighting fair, of course. None accept to fight more than a pair o' heavy men-o-war, some others, like me, still prefer fighting with a single vessel. Like me, they will wish to fight to test the strength of yer exotic technologies or strategies, savvy?"`

Reviewer 2: He can still prefers it even if it's not what he just did, so I don't think this needs further branching.

> 			`	"If ye be feeling up to the challenge, there be other captains that would be willing to duel ye, if ye keep fighting fair, of course. None accept to fight more than a pair o' heavy men-o-war, some others, like me, still prefer fighting with a single vessel. Like me, they will wish to fight to test the strength of yer exotic technologies or strategies, savvy?"`

Author: I can just add a "when appropriate"?

> 			``

Author: it's to space the thing so you have a higher chance of noticing what he says after, bc its important

			`	"Now land at once, we must officially conclude this duel, matey, else it will be worth no more than an empty chest!"`
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail If Killed"
	on visit
		dialog "You have not yet defeated the Unfettered Captain, and landing like this means you choose to delay the duel to another time."
	on complete
		conversation
			`You land and the Captain joins you once he is done with repairs. He presents what looks like an aquarium as a reward for the duel.`

> 			`	As you move closer to it to inspect it you are almost frightened to death by the huge scorpion looking creature rushing towards you.`

Reviewer 2: ```suggestion
			`	As you move closer to inspect it you are almost frightened to death by the huge scorpion looking creature rushing towards you.`
```

No need for that repitition.

			`	There is little time to think as the huge creature gets closer and closer!`
			choice
				`	(Run far away in case it escapes!)`
					goto run
				`	(Leap back to be at a respectable distance!)`
				`	(Stay there.)`
					goto stay
				`	(Shoot it.)`
					goto shoot

> 			`	Surprisingly, no Unfettered look amused by the apparent cowardiness. "Fear is good, so long as it does not control you," one of them says.`

Reviewer 2: This was likely meant to be "cowardliness," which is indeed correct.

Cowardice would also be correct here, though there are differences in implication ("being"
versus "being like") between the two.

> 			`	Surprisingly, no Unfettered look amused by the apparent cowardiness. "Fear is good, so long as it does not control you," one of them says.`

Reviewer 7: `cowardliness` is a bit weird imo, but it would be correct. `cowardiness` is not correct and should be changed.

			label reward
			action
				outfit "Heshi"

> 			`	Si-Ra does not give you any explanation, instead he proceeds to raise his fist. As you do not react he gestures with his other hand, clearly meaning you ought to do the same.`

Author: ~~right, the captain~~

			`	After you rise your fist in response, he bows his head so slightly you barely notice it, and moves away. You wonder what you should do with the creature, but refusing the gift does not seem like an option.`
			label end
			action
				log `Defeated a young Unfettered Captain in a duel, and took part in a surprisingly scary Unfettered martial ceremony.`
				event "unfettered: JD raid plan ready" 5 15
				set "unfettered: duel ready"
				# there's no way to get these before the duel
				"unfettered: took Si-Ra' stuff" = "outfit (all): Gold Bars" + "outfit (all): Ionic Blaster Prototype" * 2
			`	As he passes near you he quickly speaks, "It'll be a few weeks before I'm ready for more action."`
				accept
			label stay
			`	You stay there, surely impressing all Hai present! However it also means you get hit by the tail of the monster as it jumps out to hit you, and everything gets dark in a minute.`

> 			`	You hear voices around you, someone screaming for help but also Si-Ra speaking calmly, "There is no cure for humans," as you hear this you sink deeper and deeper, almost losing consciousness, "there is nothing to be done. This is a fight they will have to win on their own."`

Reviewer 2: ```suggestion
			`	You hear voices around you, someone screaming for help, but also your recent opponent speaking calmly. "There is no cure for humans," you hear as you sink deeper and deeper, down to the edge of unconsciousness, "there is nothing to be done. This is a fight they will have to win on their own."`
```

Missing comma, and touching up the rest, with the above ntoe in mind.

			`	You are taken far away by a beam of pure light. You see collapsed civilisations, fighting to reclaim their forgotten glory as they struggle even to feed their own people. You see a planet, once beautiful, get bombarded until not a single trace of life is left. You see stars, beautiful and seemingly eternal, get blasted away with all surrounding space. You see automated ships, repairing wounds.`
			action
				set "unfettered: heshi trip"
				"unfettered rep: Si-Ra" ++
			`	Until, finally you are brought back by that same beam of light, and it is all you can see.`
			choice
				`	(Wake up.)`
				`	(Give up.)`
					die
			`	You wake up in a hospital bed, Si-Ra is there and speaks very quietly, almost gently, "That was maybe not smart, lad, but it sure was brave! You have what it takes to be one of us, just make sure you don't get killed next time." He then moves to the door behind you.`
				goto end
			label run
			action
				log "Defeated a young Unfettered Captain in a duel, but it seems that was not the whole test. The second part involved a monstruous scorpion from which it felt like a good idea to run at the time. The Unfettered did not see it that way, however."

> 			`	You start running away and hear Si-Ra speak to his crew, loud enough for you to hear, "The monkey is running away from danger. It would not be fit to join us indeed. It will not fight for us."`

Reviewer 2: Good catch on this being placed out of order to refer to him by name, but I'd prefer keeping that reveal for later and rewording the references in this segment.

This whole section could stand a general style pass anyway, ("hear him speak to his crew, loud enough for you to hear" in this paragraph, for example) so feel free, if you're interested, to put forward suggestions to that regard in line with your pass on the sword description :)

				decline
			label shoot
			`	You get your weapon out and shoot the monster as it attempts to rush out of its aquarium. You are not sure if you hit it with the erupting water that came out your way as the beast emerged, but at least you are still alive!`

> 				goto reward

Reviewer 2: I agree.

I get the impression we're meant to have not directly hit it, anyway, and even that only after it's left the water, but this line especially could use sprucing up in general. especially if we really are essentially firing off into the crowd wildly lol

Perhaps we do hit it, and the strange Hai captain gifts us another in it's place, praising the reaction speed and aim in?

> 				goto reward

Author: idea is you shot over the aquarium as the monster emerges, and it made it stay in

> 				goto reward

Reviewer 2: Yea, that could definitely use some clarifying/follow up in a style pass of this section :)

			choice
				`	"What can I do for my favorite squirrel?"`
					goto insult
				`	"How soon will you have that new ship?"`
				`	"Well, just do it with that bigger ship of yours, then. I'm not interested."`
					decline
			`	"Once this job be done, hopefully! It should pay handsomely - 'buying a brand new ship' kind of handsome." Judging from your last encounter, this will probably be a very dangerous mission, but it may well be worth it.`
			choice
				`	"What kind of job is it?"`
					goto job
			label insult
			`	He does not seem to react to the insult. He did call you monkey a few times before, so it's only fair for you to return the courtesy.`
			label job
			`	"We be going to collect a few jump drives! Normally ye would not be able to get such a job, since ye have not earned much among us still, but I have the connection to get it for ye. Ye see, officially this be me mission, and a very lucrative one, at that! On top of that, should we succeed, next time ye may be entrusted with such missions on your own, but don't ye be expecting near the same booty for yerself."`
			choice
				`	"Okay... That just seems too good to be true. What's the catch?"`

> 			`	"Well... the world be called 'Freedom', and that may sound nice, but it be a dangerous human port, full of privateers and swashbucklers. That is why I need yer ship, for transportation beyond Hai space. Ye will also need to save room in yer cargo hold or on your flagship for four jump drives. Ye be warned: this be no mission for the lighthearted."`

Author: what about "be warned:"
?

			choice
				`	"Let's go then!"`
					goto leave
				`	"Why can't you just use a jump drive and go there yourself?"`
				`	"What will my cut be?"`
					goto cut
			action
				"unfettered rep: Si-Ra" --

> 			`	"Jump drives be a fine spot of loot amongst me people. I would not risk one by traveling through hostile space. Ye've had one yerself, so ye should know about the scurvy dogs to the south who give no quarter. The lily-livered scallywags to the north would like to see me walk the plank as well... A jump drive would be of no special help to us on this voyage." He seems disappointed by you having to ask such an obvious thing.`

Author: it's fine but mb you should get an opportunity to ask him about it

			has "language: Hai"
		<secondgroup> "He does not seem pleased, speaking with contained anger. You are not sure if he is demanding something, or simply complaining."
			not "language: Hai"

> 		<secondgroup> `"Things are going [poorly, worse]. In all my life never has it been more [rough, unsustainable]. Never have there been so many in need! Never so little to harvest, worse [housing, sanitation]... Something needs to be done, and quickly! We, and our way of life, are dying."`

Reviewer 6: I feel the exclamation points are needed to convey.  `...speaking with contained anger` alternate dialogue from above. 
But, like you Hurl, I'm not an expert in writing.

			label translate
			action
				log "Minor People" "Merora" "A young adolescent Hai that came to Unfettered space after losing her parents to pirates. She is handling the loss very well, at least on the surface."

> 			`	"Great! I am Merora, by the way." She looks around, noticing the way some people are staring at you. Surprisingly they seem to be staring at her almost as much as they do at you. She explains, "I came to Unfettered space when I had nowhere left to turn to, just a few weeks ago. I don't feel like I belong here yet, or anywhere for that matter. So trust me, I know what it feels like to be isolated here. What about you," she asks.`

Reviewer: ```suggestion
			`	"Great! I am Merora, by the way." She looks around, noticing the way some people are staring at you. Surprisingly they seem to be staring at her almost as much as they do at you. She explains, "I came to Unfettered space when I had nowhere left to turn to, just a few weeks ago. I don't feel like I belong here yet, or anywhere for that matter. So trust me, I know what it feels like to be isolated here. What about you?" she asks.`
```
~~not sure, but the other version looks weird~~

					goto brethren
				`	"Why do you need me to do it?"`
			label whyme
			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high, due to an increase in the amount of Hai wanting to come to our space. Even in normal times, the Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not that many Hai captains are willing to come here. When they do, some charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Others are more generous, of course, but it is still not enough.`

> 			`	Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`

Reviewer: ```suggestion
			`	"Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
```
Not a huge fan of this tbh.

Author: wdym? you think he should say smth else or?

Reviewer: The first sentence feels a bit too long and convoluted. It's not much of a problem since an alien says it, you can keep it if you want to.

Author: I think it's fine, I don't want him to have perfect english anyhow, in fact its good if he slips a bit
still, thx for the input!

				`	(Run away.)`
					goto run
			label crush
			action
				"unfettered rep: Sira" --

> 			`	You rush his way, fist raised, giving his friends in the crowd the exact excuse they were looking for to open fire on you, but thankfully they show restrain as he shouts to command them to stop. You sense your right arm is numb from the shots, but other than that you are fine.`

Reviewer: ```suggestion
			`	You rush his way, fist raised, giving his friends in the crowd the exact excuse they were looking for to open fire on you, but thankfully they show restrain as he shouts for them to stop. You sense your right arm go numb from the shots, but other than that you are fine.`
```

> 			`	"Ye think yerself strong, eh mate? You better be ready to prove it in a proper duel with your ship, against me and me mateys!" He shouts, but he seems more disapointed than angry.`

Reviewer: ```suggestion
			`	"Ye think yerself strong, eh mate? Ye better be ready to prove it in a proper duel with your ship, against me and me mateys!" He shouts, but he seems more disappointed than angry.`
```

> 			`	Then he moves with frightnening speed, and somehow is in front of you with a blade on your neck before you can do anything about it. "And don't test me," he adds with a dark look, before putting his blade away once he's sure he got his point across. The crowd stays silent but you can tell they were prepared the support him, had the need arrisen.`

Reviewer: ```suggestion
			`	He then moves with frightening speed, and is somehow in front of you with a blade on your neck before you could move a muscle. "And don't test me," he adds with a dark look, before putting his blade away once he's sure he got his point across. The crowd stays silent, but you can tell they were prepared to support him, had the need arisen.`
```

			label fire
			`	In the background you hear fires roaring, it seems the damage to his ship was quite serious indeed.`
			choice

> 				`	"Be careful with provoking men you may have gotten killed! And don't count on me to repair your ship."`

Reviewer: > Be careful with provoking men

1. We don't specify FL's gender
2. ~~why~~

Author: bruh this was not intended

					goto cut
			action
				"unfettered rep: Si-Ra" --
			`	"Jump drives be a fine spot of loot amongst me people. I would not risk one by traveling through hostile space. Ye've had one yerself, so ye should know about the scurvy dogs to the south who give no quarter. The parrots to the north would like to see me walk the plank as well... A jump drive would be of no special help to us on this voyage." He seems disappointed by you having to ask such an obvious thing.`
			choice
				`	"Let's go then!"`
					goto leave
				`	"parrots?"`
					goto aliens
				`	"Sorry, but I won't risk my ship by going anywhere close to these pirates."`
					decline
			label cut
			`	"Ye humans always be obsessed with yer doubloons... So many much greater a variety of booty ye could be seeking. Rest assured, it be quite the coffer. I be taking half of the total bounty, and ye should well remember how well ye were paid for the ones ye've already delivered to me brethren."`
			label leave

> 			`	"One last warning for ye: Ye should sail the skies with a hyperdrive, especially with the dangers of where we be headed. A jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk whether or not the lily-livered scallywags up north would decide to fit ye with a hempen halter from yer association with us."`

Reviewer: ```suggestion
			`	"One last warning for ye: Ye should sail the skies with a hyperdrive, especially with the dangers of where we be headed. A jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk whether or not the parrots up north would decide to fit ye with a hempen halter from yer association with us."`
```
This is how the text below makes sense

			label cut
			`	"Ye humans always be obsessed with yer doubloons... So many much greater a variety of booty ye could be seeking. Rest assured, it be quite the coffer. I be taking half of the total bounty, and ye should well remember how well ye were paid for the ones ye've already delivered to me brethren."`
			label leave

> 			`	"One last warning for ye: Ye should sail the skies with a hyperdrive, especially with the dangers of where we be headed. A jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk whether or not the perrots up north would decide to fit ye with a hempen halter from yer association with us."`

Reviewer: ```suggestion
			`	"One last warning for ye: Ye should sail the skies with a hyperdrive, especially with the dangers of where we be headed. A jump drive won't do us any favors getting there, but 'twill bring more than a mite of unwanted company. Nor should we risk whether or not the parrots up north would decide to fit ye with a hempen halter from yer association with us."`
```
~~I did _not_ suggest `perrot`, I checked.~~

			`	"We be not the ones who placed these drives here..." The Hai doesn't seem to want to tell you much about this vault.`
			label ship
			`	As soon as your ship arrives, you give the orders to collect the jump drives and install them if possible, so you may flee more swiftly. When your ship takes off, you notice more activity than on the way in...`

> 			`	The Hai comes to speak to you on the bridge. "Batten down the hatches, matey! It took us too long to check and load the drives, and we seem to have attracted attention. I guess a lookout got away from the vault and contacted some friends..." Multiple alarms start going off, confirming his suspicions - hostile ships are launching and heading straight for you. You launch as soon as you can, rushing through the correct procedures. Meanwhile, the Unfettered captain joins you on the bridge, clearly used to being where the action is, invited or not, and gives you advice, "They'll probably be launching after us, you should first get some distance from them before trying to jump away."`

Reviewer: ```suggestion
			`	The Hai comes to speak to you on the bridge. "Batten down the hatches, matey! It took us too long to check and load the drives, and we seem to have attracted attention. I guess a lookout got away from the vault and contacted some friends..." Multiple alarms start going off, confirming his suspicions - hostile ships are launching and heading straight for you. You launch as soon as you can, rushing through the correct procedures. Meanwhile, the Unfettered captain joins you on the bridge, clearly used to being where the action is, invited or not, giving you advice. "They'll probably be launching after us, you should first get some distance from them before trying to jump away."`
```

					goto finalmention
				`	"I will find them and bring the loot back, whatever the cost.`
			label whatevercost
			`	He seems a bit warry of your attitude, "Please, whatever you do, try to avoid needless bloodshed, these pirates are not our enemies. I would hardly recommend bribing those that can be. That said, I expect you to do what you must in order to bring back all 142 tons of materials, you can either fit all of that on a Shield Beetle, or bring them back in multiple trips, by storing them in an outfitter or on your other ships. And remember to look for it north-west of human space."`
			label finalmention
			branch nvm
				has "flagship model: Shield Beetle"
			`	"You must know of the current food situation we have. Far from me to suggest you should profit from it, but I know one can easily make a lot of money moving cargo around in aphid, especially if they do not risk being attack by us, ha!"`
				to display
					has "unfettered: hunger"
					# cost of a Shield Beetle
					credits < 32000000
			# the stock SB has barely enough cargo to fit the 142 tons
			`	"I would advise you to lead this mission a Shield Beetle, like a true warrior. You deserve it."`

> 			label nvm

Author: to skip over the things just above it
I could do it with to display but its clearer this way I think

> 			label nvm

Reviewer: Can't you just `accept` where you currently `goto nvm`? This is the last label, and it's empty, no need to keep it going.

> 			label nvm

Author: I could do a node that would decline and use a to display but that'd be hacky I think

			label intention
			`	"That's none of your business, but if it can help you get away... My Masters are coming soon to retrieve the cargo, and if it ain't there no more, I'll be the one they 'retrieve' instead. So you better forget about it!" He says, before abruptly closing the communication.`
			label over

> 			`	`

Author: I'm using it to space things

					goto agree
				`	"How do I scan it?"`
			label scan
			`	"You need to use one of our Value Detectors, the information provided by it should be sufficient. If you don't have one, you can install one on Darkcloak, on the way there. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`
			choice
				`	"Sure, will do."`
					goto agree
				`	"I'm not interested in scanning ships for you."`
					decline
			label agree
			`	On the way back to your ship, Mera runs up to you. "One... last... thing, captain!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you one of our scanners. It will be placed in your cargo hold if there is no space on your ship."`

> 			`	I cannot stress enough that you NEED to use this scanner, as any other will simply either not contain enough information, or we could be unable to make good use of it.`

Reviewer: ```suggestion
			`	"I cannot stress enough that you NEED to use this scanner, as any other will simply either not contain enough information, or we could be unable to make good use of it."`
```

				`	"But I can jump away without risk."`
					goto jump
				`	"Screw you and your Chiefs! I'm taking that ship down!"`
				`	"But I have an urgent mission for one of your engineers."`
					goto mission
			`	"Consider this carefully before taking any actions you may regret in the future. Our offer still stands." The transmission cuts out.`
				decline
			label jump
			`	"Maybe, but there is no guarantee their ship will not pursue you. Please, land straight away, I assure you the other ship will land on another planet, and if not, our orbital defences will protect you."`
				decline
			label mission

> 			`	"Yes, we know. You do not need to worry about that, whilst we will not relay the scans ourselves for safety reasons, bringing them back never was the urgency, you may do so whenever you have the time."`

Author: ~~but how?~~
I really tried but idk how to word it right
this is a random unfettered hai spaceport dude tho so if they speak bad its good x)

> 			`	"Yes, we know. You do not need to worry about that, whilst we will not relay the scans ourselves for safety reasons, bringing them back never was the urgency, you may do so whenever you have the time."`

Reviewer: I guess it's fine then, aliens don't speak English.

	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on complete
		dialog
			`You are offered some surprisingly high-quality beverages to keep you waiting, even by your standards. This special treatment is probably in part due to your ongoing mission for Mera.`
			`	After a while, the ship goes away, you can now go back to <origin> with the scanner logs.`

mission "Unfettered: I Know The Way"
	landing
	invisible
	to offer
		has "Unfettered: Solifuge 2: Auxiliary Design: done"
	on enter "Host"
		dialog "Upon entering the system you realize it is very quiet, unlike most nearby systems. This would appear to be the ideal place to hide..."
	destination "Avalon"
	to fail
		has "unfettered: it's the Alphas!"
	on visit
		dialog "You suspect there is something here, but after having done a full sweep of the planet your sensors reveal nothing."
	to complete
		# You must have some kind of scanner to find it, the tactical scanner is not enough. The living ships are able to find it.
		"outfit (installed): Value Detector" + "outfit (installed): Research Laboratory" + "outfit (installed): Surveillance Pod" + "ship model: Peregrine" + "ship model: Merganser" > 0
	on complete

> 		dialog "It takes you a lot of time, but suspecting there must be something to find here you persevere, until you find a very well conceiled Alpha base, using your ship's sensors. This must be where the Auxiliary came from."

Reviewer: ```suggestion
		dialog "It takes you a lot of time, but suspecting there must be something to find here you persevere, until you find a very well concealed Alpha base using your ship's sensors. This must be where the Auxiliary came from."
```

		set "unfettered: it's the Alphas!"

# if you betrayed Midas, you will suffer to find answers
mission "Unfettered: Questions Best Left Unanswered"
	landing
	invisible
	to offer
		has "Unfettered: Solifuge 2: Auxiliary Design: done"
	on enter "Almaaz" "Arneb" "Alninam"
		dialog "As you enter the system it occurs to you the nearby pirates may have information regarding that Auxiliary."
	on enter "Alcyone" "Misam"

> 		dialog "Entering the system you recall that the pirates of Misam seemed to be in the business of knowing things."

Reviewer: > in the business of knowing things

idk if this fits the narrative style

	to fail
		has "unfettered: it's the Alphas!"

mission "Unfettered: No Answers Available"
	landing
	invisible
	to offer
		has "Unfettered: Questions Best Left Unanswered: active"
	source
		government "Pirate"
	on offer
		conversation
			branch Misam
				has "flagship planet: New Tortuga"
			`You ask questions around but nobody knows anything about an Auxiliary here, or at least they're not willing to tell you. You may have more luck at another planet.`
				defer
			label Misam
			branch "okay boy!"
				not "midas: wronged"
			branch "dead, boy!"
				"midas: wronged" > 1

> 			`You ask around for this Auxiliary; 'Settler', but no-one will reveal anything about it to you, until, suspecting you will not leave until you have something, someone comes, their face conceiled. "You. You have betrayed us by revealing our plan to the governor, you are not to be trusted. That said, those that you are looking for are our enemies as well, so I will reveal you the location you need -if- you agree to give us compensation for your mistake. We demand 10 millions, in the form of a fine, to make sure you will have to pay us interests should you lack money."`

Reviewer: ```suggestion
			`You ask around for this Auxiliary 'Settler', but no-one will reveal anything about it to you, until, suspecting you will not leave until you have something, someone comes, their face concealed. "You. You have betrayed us by revealing our plan to the governor, you are not to be trusted. That said, those that you are looking for are our enemies as well, so I will reveal you the location you need -if- you agree to give us compensation for your mistake. We demand 10 millions, in the form of a fine, to make sure you will have to pay us interests should you lack money."`
```

			choice
				`	"I will give you your 10 millions.`
				`	"I refuse. I will find my own way to find who is in control of that ship, however hard that may be."`
					decline
			action
				fine 10000000
			`	Upon verification of the payment, you receive a set of coordinates on this world.`
			label "okay boy!"
			action
				set "unfettered: it's the Alphas!"
			`You head to the given coordinates on the planet, but you cannot see anyone there. You wait around for an hour or so until, looking around, you finally spot it: there is a graffiti on the wall, right at the given coordinate. At first you did not pay too much attention to it because this kind of thing is common on pirate worlds, but now you notice it depicts a green human. The message is clear.`
				decline
			label "dead, boy!"

> 			`As soon as you set foot on the planet, a team of mercenaries comes out of the shadows, armored with the same kind of armor Si-Ra had. Their weapons may holstered but you can feel the tension in the air. Their leader speaks up. "We know your query. Consider yourself lucky to not be killed on the spot. We do not guarantee that will not happen, should you land again."`

Reviewer: ```suggestion
			`As soon as you set foot on the planet, a team of mercenaries comes out of the shadows, armored with the same kind of armor Si-Ra had. Their weapons may be holstered, but you can feel the tension in the air. Their leader speaks up. "We know your query. Consider yourself lucky to not be killed on the spot. We do not guarantee that will not happen, should you land again."`
```

> 			`	They evaluate their surrendings before coming closer to you, and once they are a meter a way they stop, clearly speaking their words as a threat, with a pause between each of them "Nemo. Me. Impune. Laccesit."`

Reviewer: ```suggestion
			`	They evaluate their surroundings before coming closer to you, and once they are a meter away they stop, clearly speaking their words as a threat, with a pause between each of them "Nemo. Me. Impune. Laccesit."`
```
Why does it feel like you were half asleep writing this mission?

Author: ~~I have no idea what you're speaking about~~

			choice
				`	(Threaten them).`
				`	"Surely there is something you can tell me?"`
					goto hint
				`	(Leave).`
					decline

> 			`	You move your arm to your weapon but before you can even begin to reach it, you find yourself shot from all directions by what appears to be ion based weapons, not unlike the Hai pulse rifles. You may have been able to survive a few shots, but given the sheer amount of fire from all directions, you never wake up again.`

Reviewer: `begin to reach it` feels weird.
